{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "652b6506",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/3dmoljs_load.v0": "<div id=\"3dmolviewer_17563213462829392\"  style=\"position: relative; width: 400px; height: 300px;\">\n        <p id=\"3dmolwarning_17563213462829392\" style=\"background-color:#ffcccc;color:black\">3Dmol.js failed to load for some reason.  Please check your browser console for error messages.<br></p>\n        </div>\n<script>\n\nvar loadScriptAsync = function(uri){\n  return new Promise((resolve, reject) => {\n    //this is to ignore the existence of requirejs amd\n    var savedexports, savedmodule;\n    if (typeof exports !== 'undefined') savedexports = exports;\n    else exports = {}\n    if (typeof module !== 'undefined') savedmodule = module;\n    else module = {}\n\n    var tag = document.createElement('script');\n    tag.src = uri;\n    tag.async = true;\n    tag.onload = () => {\n        exports = savedexports;\n        module = savedmodule;\n        resolve();\n    };\n  var firstScriptTag = document.getElementsByTagName('script')[0];\n  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n});\n};\n\nif(typeof $3Dmolpromise === 'undefined') {\n$3Dmolpromise = null;\n  $3Dmolpromise = loadScriptAsync('https://cdn.jsdelivr.net/npm/3dmol@2.5.2/build/3Dmol-min.js');\n}\n\nvar viewer_17563213462829392 = null;\nvar warn = document.getElementById(\"3dmolwarning_17563213462829392\");\nif(warn) {\n    warn.parentNode.removeChild(warn);\n}\n$3Dmolpromise.then(function() {\nviewer_17563213462829392 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_17563213462829392\"),{backgroundColor:\"white\"});\nviewer_17563213462829392.zoomTo();\n\tviewer_17563213462829392.addModel(\"HEADER    HORMONE                                 01-JUL-09   3I40              \\nTITLE     HUMAN INSULIN                                                         \\nCOMPND    MOL_ID: 1;                                                            \\nCOMPND   2 MOLECULE: INSULIN A CHAIN;                                           \\nCOMPND   3 CHAIN: A;                                                            \\nCOMPND   4 ENGINEERED: YES;                                                     \\nCOMPND   5 MOL_ID: 2;                                                           \\nCOMPND   6 MOLECULE: INSULIN B CHAIN;                                           \\nCOMPND   7 CHAIN: B;                                                            \\nCOMPND   8 ENGINEERED: YES                                                      \\nSOURCE    MOL_ID: 1;                                                            \\nSOURCE   2 ORGANISM_SCIENTIFIC: HOMO SAPIENS;                                   \\nSOURCE   3 ORGANISM_COMMON: HUMAN;                                              \\nSOURCE   4 ORGANISM_TAXID: 9606;                                                \\nSOURCE   5 EXPRESSION_SYSTEM: ESCHERICHIA COLI;                                 \\nSOURCE   6 EXPRESSION_SYSTEM_TAXID: 562;                                        \\nSOURCE   7 MOL_ID: 2;                                                           \\nSOURCE   8 ORGANISM_SCIENTIFIC: HOMO SAPIENS;                                   \\nSOURCE   9 ORGANISM_COMMON: HUMAN;                                              \\nSOURCE  10 ORGANISM_TAXID: 9606;                                                \\nSOURCE  11 EXPRESSION_SYSTEM: ESCHERICHIA COLI;                                 \\nSOURCE  12 EXPRESSION_SYSTEM_TAXID: 562                                         \\nKEYWDS    INSULIN, CARBOHYDRATE METABOLISM, CLEAVAGE ON PAIR OF BASIC RESIDUES, \\nKEYWDS   2 DIABETES MELLITUS, DISEASE MUTATION, DISULFIDE BOND, GLUCOSE         \\nKEYWDS   3 METABOLISM, HORMONE, SECRETED                                        \\nEXPDTA    X-RAY DIFFRACTION                                                     \\nAUTHOR    V.I.TIMOFEEV,V.V.BEZUGLOV,K.A.MIROSHNIKOV,R.N.CHUPROV-NETOCHIN,       \\nAUTHOR   2 I.P.KURANOVA                                                         \\nREVDAT   4   06-NOV-24 3I40    1       REMARK                                   \\nREVDAT   3   01-NOV-23 3I40    1       REMARK                                   \\nREVDAT   2   05-MAY-10 3I40    1       JRNL                                     \\nREVDAT   1   05-JAN-10 3I40    0                                                \\nJRNL        AUTH   V.I.TIMOFEEV,R.N.CHUPROV-NETOCHIN,V.R.SAMIGINA,V.V.BEZUGLOV, \\nJRNL        AUTH 2 K.A.MIROSHNIKOV,I.P.KURANOVA                                 \\nJRNL        TITL   X-RAY INVESTIGATION OF GENE-ENGINEERED HUMAN INSULIN         \\nJRNL        TITL 2 CRYSTALLIZED FROM A SOLUTION CONTAINING POLYSIALIC ACID.     \\nJRNL        REF    ACTA CRYSTALLOGR.,SECT.F      V.  66   259 2010              \\nJRNL        REFN                   ESSN 1744-3091                               \\nJRNL        PMID   20208155                                                     \\nJRNL        DOI    10.1107/S1744309110000461                                    \\nREMARK   2                                                                      \\nREMARK   2 RESOLUTION.    1.85 ANGSTROMS.                                       \\nREMARK   3                                                                      \\nREMARK   3 REFINEMENT.                                                          \\nREMARK   3   PROGRAM     : PHENIX (PHENIX.REFINE)                               \\nREMARK   3   AUTHORS     : PAUL ADAMS,PAVEL AFONINE,VINCENT CHEN,IAN            \\nREMARK   3               : DAVIS,KRESHNA GOPAL,RALF GROSSE-KUNSTLEVE,           \\nREMARK   3               : LI-WEI HUNG,ROBERT IMMORMINO,TOM IOERGER,            \\nREMARK   3               : AIRLIE MCCOY,ERIK MCKEE,NIGEL MORIARTY,              \\nREMARK   3               : REETAL PAI,RANDY READ,JANE RICHARDSON,               \\nREMARK   3               : DAVID RICHARDSON,TOD ROMO,JIM SACCHETTINI,           \\nREMARK   3               : NICHOLAS SAUTER,JACOB SMITH,LAURENT                  \\nREMARK   3               : STORONI,TOM TERWILLIGER,PETER ZWART                  \\nREMARK   3                                                                      \\nREMARK   3    REFINEMENT TARGET : ML                                            \\nREMARK   3                                                                      \\nREMARK   3  DATA USED IN REFINEMENT.                                            \\nREMARK   3   RESOLUTION RANGE HIGH (ANGSTROMS) : 1.85                           \\nREMARK   3   RESOLUTION RANGE LOW  (ANGSTROMS) : 31.53                          \\nREMARK   3   MIN(FOBS/SIGMA_FOBS)              : 0.000                          \\nREMARK   3   COMPLETENESS FOR RANGE        (%) : 99.1                           \\nREMARK   3   NUMBER OF REFLECTIONS             : 6640                           \\nREMARK   3                                                                      \\nREMARK   3  FIT TO DATA USED IN REFINEMENT.                                     \\nREMARK   3   R VALUE     (WORKING + TEST SET) : 0.202                           \\nREMARK   3   R VALUE            (WORKING SET) : 0.200                           \\nREMARK   3   FREE R VALUE                     : 0.228                           \\nREMARK   3   FREE R VALUE TEST SET SIZE   (%) : 4.760                           \\nREMARK   3   FREE R VALUE TEST SET COUNT      : 316                             \\nREMARK   3                                                                      \\nREMARK   3  FIT TO DATA USED IN REFINEMENT (IN BINS).                           \\nREMARK   3   BIN  RESOLUTION RANGE  COMPL.    NWORK NFREE   RWORK  RFREE        \\nREMARK   3     1 31.5295 -  2.3309    1.00     3219   154  0.1910 0.2145        \\nREMARK   3     2  2.3309 -  1.8501    0.99     3105   162  0.2301 0.2627        \\nREMARK   3                                                                      \\nREMARK   3  BULK SOLVENT MODELLING.                                             \\nREMARK   3   METHOD USED        : FLAT BULK SOLVENT MODEL                       \\nREMARK   3   SOLVENT RADIUS     : 1.11                                          \\nREMARK   3   SHRINKAGE RADIUS   : 0.90                                          \\nREMARK   3   K_SOL              : 0.35                                          \\nREMARK   3   B_SOL              : 47.05                                         \\nREMARK   3                                                                      \\nREMARK   3  ERROR ESTIMATES.                                                    \\nREMARK   3   COORDINATE ERROR (MAXIMUM-LIKELIHOOD BASED)     : 0.240            \\nREMARK   3   PHASE ERROR (DEGREES, MAXIMUM-LIKELIHOOD BASED) : 24.930           \\nREMARK   3                                                                      \\nREMARK   3  B VALUES.                                                           \\nREMARK   3   FROM WILSON PLOT           (A**2) : 24.04                          \\nREMARK   3   MEAN B VALUE      (OVERALL, A**2) : 30.30                          \\nREMARK   3   OVERALL ANISOTROPIC B VALUE.                                       \\nREMARK   3    B11 (A**2) : NULL                                                 \\nREMARK   3    B22 (A**2) : NULL                                                 \\nREMARK   3    B33 (A**2) : NULL                                                 \\nREMARK   3    B12 (A**2) : NULL                                                 \\nREMARK   3    B13 (A**2) : NULL                                                 \\nREMARK   3    B23 (A**2) : NULL                                                 \\nREMARK   3                                                                      \\nREMARK   3  TWINNING INFORMATION.                                               \\nREMARK   3   FRACTION: NULL                                                     \\nREMARK   3   OPERATOR: NULL                                                     \\nREMARK   3                                                                      \\nREMARK   3  DEVIATIONS FROM IDEAL VALUES.                                       \\nREMARK   3                 RMSD          COUNT                                  \\nREMARK   3   BOND      :  0.005            423                                  \\nREMARK   3   ANGLE     :  0.760            569                                  \\nREMARK   3   CHIRALITY :  0.068             62                                  \\nREMARK   3   PLANARITY :  0.002             73                                  \\nREMARK   3   DIHEDRAL  : 17.205            141                                  \\nREMARK   3                                                                      \\nREMARK   3  TLS DETAILS                                                         \\nREMARK   3   NUMBER OF TLS GROUPS  : NULL                                       \\nREMARK   3                                                                      \\nREMARK   3  NCS DETAILS                                                         \\nREMARK   3   NUMBER OF NCS GROUPS : NULL                                        \\nREMARK   3                                                                      \\nREMARK   3  OTHER REFINEMENT REMARKS: NULL                                      \\nREMARK   4                                                                      \\nREMARK   4 3I40 COMPLIES WITH FORMAT V. 3.30, 13-JUL-11                         \\nREMARK 100                                                                      \\nREMARK 100 THIS ENTRY HAS BEEN PROCESSED BY PDBJ ON 06-JUL-09.                  \\nREMARK 100 THE DEPOSITION ID IS D_1000053935.                                   \\nREMARK 200                                                                      \\nREMARK 200 EXPERIMENTAL DETAILS                                                 \\nREMARK 200  EXPERIMENT TYPE                : X-RAY DIFFRACTION                  \\nREMARK 200  DATE OF DATA COLLECTION        : 10-MAR-09                          \\nREMARK 200  TEMPERATURE           (KELVIN) : 100                                \\nREMARK 200  PH                             : NULL                               \\nREMARK 200  NUMBER OF CRYSTALS USED        : 1                                  \\nREMARK 200                                                                      \\nREMARK 200  SYNCHROTRON              (Y/N) : N                                  \\nREMARK 200  RADIATION SOURCE               : ROTATING ANODE                     \\nREMARK 200  BEAMLINE                       : NULL                               \\nREMARK 200  X-RAY GENERATOR MODEL          : BRUKER AXS MICROSTAR               \\nREMARK 200  MONOCHROMATIC OR LAUE    (M/L) : M                                  \\nREMARK 200  WAVELENGTH OR RANGE        (A) : NULL                               \\nREMARK 200  MONOCHROMATOR                  : NULL                               \\nREMARK 200  OPTICS                         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  DETECTOR TYPE                  : CCD                                \\nREMARK 200  DETECTOR MANUFACTURER          : MAR CCD 165 MM                     \\nREMARK 200  INTENSITY-INTEGRATION SOFTWARE : DENZO                              \\nREMARK 200  DATA SCALING SOFTWARE          : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  NUMBER OF UNIQUE REFLECTIONS   : 6677                               \\nREMARK 200  RESOLUTION RANGE HIGH      (A) : 1.850                              \\nREMARK 200  RESOLUTION RANGE LOW       (A) : 54.636                             \\nREMARK 200  REJECTION CRITERIA  (SIGMA(I)) : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 OVERALL.                                                             \\nREMARK 200  COMPLETENESS FOR RANGE     (%) : NULL                               \\nREMARK 200  DATA REDUNDANCY                : NULL                               \\nREMARK 200  R MERGE                    (I) : NULL                               \\nREMARK 200  R SYM                      (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR THE DATA SET  : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 IN THE HIGHEST RESOLUTION SHELL.                                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE HIGH (A) : NULL                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE LOW  (A) : NULL                     \\nREMARK 200  COMPLETENESS FOR SHELL     (%) : NULL                               \\nREMARK 200  DATA REDUNDANCY IN SHELL       : NULL                               \\nREMARK 200  R MERGE FOR SHELL          (I) : NULL                               \\nREMARK 200  R SYM FOR SHELL            (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR SHELL         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 DIFFRACTION PROTOCOL: SINGLE WAVELENGTH                              \\nREMARK 200 METHOD USED TO DETERMINE THE STRUCTURE: MOLECULAR REPLACEMENT        \\nREMARK 200 SOFTWARE USED: NULL                                                  \\nREMARK 200 STARTING MODEL: PDB ENTRY 2G4M                                       \\nREMARK 200                                                                      \\nREMARK 200 REMARK: NULL                                                         \\nREMARK 280                                                                      \\nREMARK 280 CRYSTAL                                                              \\nREMARK 280 SOLVENT CONTENT, VS   (%): 62.90                                     \\nREMARK 280 MATTHEWS COEFFICIENT, VM (ANGSTROMS**3/DA): 3.31                     \\nREMARK 280                                                                      \\nREMARK 280 CRYSTALLIZATION CONDITIONS: NULL                                     \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY                                            \\nREMARK 290 SYMMETRY OPERATORS FOR SPACE GROUP: I 21 3                           \\nREMARK 290                                                                      \\nREMARK 290      SYMOP   SYMMETRY                                                \\nREMARK 290     NNNMMM   OPERATOR                                                \\nREMARK 290       1555   X,Y,Z                                                   \\nREMARK 290       2555   -X+1/2,-Y,Z+1/2                                         \\nREMARK 290       3555   -X,Y+1/2,-Z+1/2                                         \\nREMARK 290       4555   X+1/2,-Y+1/2,-Z                                         \\nREMARK 290       5555   Z,X,Y                                                   \\nREMARK 290       6555   Z+1/2,-X+1/2,-Y                                         \\nREMARK 290       7555   -Z+1/2,-X,Y+1/2                                         \\nREMARK 290       8555   -Z,X+1/2,-Y+1/2                                         \\nREMARK 290       9555   Y,Z,X                                                   \\nREMARK 290      10555   -Y,Z+1/2,-X+1/2                                         \\nREMARK 290      11555   Y+1/2,-Z+1/2,-X                                         \\nREMARK 290      12555   -Y+1/2,-Z,X+1/2                                         \\nREMARK 290      13555   X+1/2,Y+1/2,Z+1/2                                       \\nREMARK 290      14555   -X,-Y+1/2,Z                                             \\nREMARK 290      15555   -X+1/2,Y,-Z                                             \\nREMARK 290      16555   X,-Y,-Z+1/2                                             \\nREMARK 290      17555   Z+1/2,X+1/2,Y+1/2                                       \\nREMARK 290      18555   Z,-X,-Y+1/2                                             \\nREMARK 290      19555   -Z,-X+1/2,Y                                             \\nREMARK 290      20555   -Z+1/2,X,-Y                                             \\nREMARK 290      21555   Y+1/2,Z+1/2,X+1/2                                       \\nREMARK 290      22555   -Y+1/2,Z,-X                                             \\nREMARK 290      23555   Y,-Z,-X+1/2                                             \\nREMARK 290      24555   -Y,-Z+1/2,X                                             \\nREMARK 290                                                                      \\nREMARK 290     WHERE NNN -> OPERATOR NUMBER                                     \\nREMARK 290           MMM -> TRANSLATION VECTOR                                  \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY TRANSFORMATIONS                            \\nREMARK 290 THE FOLLOWING TRANSFORMATIONS OPERATE ON THE ATOM/HETATM             \\nREMARK 290 RECORDS IN THIS ENTRY TO PRODUCE CRYSTALLOGRAPHICALLY                \\nREMARK 290 RELATED MOLECULES.                                                   \\nREMARK 290   SMTRY1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY1   2 -1.000000  0.000000  0.000000       38.61000            \\nREMARK 290   SMTRY2   2  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   2  0.000000  0.000000  1.000000       38.61000            \\nREMARK 290   SMTRY1   3 -1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   3  0.000000  1.000000  0.000000       38.61000            \\nREMARK 290   SMTRY3   3  0.000000  0.000000 -1.000000       38.61000            \\nREMARK 290   SMTRY1   4  1.000000  0.000000  0.000000       38.61000            \\nREMARK 290   SMTRY2   4  0.000000 -1.000000  0.000000       38.61000            \\nREMARK 290   SMTRY3   4  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290   SMTRY1   5  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY2   5  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   5  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY1   6  0.000000  0.000000  1.000000       38.61000            \\nREMARK 290   SMTRY2   6 -1.000000  0.000000  0.000000       38.61000            \\nREMARK 290   SMTRY3   6  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY1   7  0.000000  0.000000 -1.000000       38.61000            \\nREMARK 290   SMTRY2   7 -1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   7  0.000000  1.000000  0.000000       38.61000            \\nREMARK 290   SMTRY1   8  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290   SMTRY2   8  1.000000  0.000000  0.000000       38.61000            \\nREMARK 290   SMTRY3   8  0.000000 -1.000000  0.000000       38.61000            \\nREMARK 290   SMTRY1   9  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   9  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY3   9  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY1  10  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2  10  0.000000  0.000000  1.000000       38.61000            \\nREMARK 290   SMTRY3  10 -1.000000  0.000000  0.000000       38.61000            \\nREMARK 290   SMTRY1  11  0.000000  1.000000  0.000000       38.61000            \\nREMARK 290   SMTRY2  11  0.000000  0.000000 -1.000000       38.61000            \\nREMARK 290   SMTRY3  11 -1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY1  12  0.000000 -1.000000  0.000000       38.61000            \\nREMARK 290   SMTRY2  12  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290   SMTRY3  12  1.000000  0.000000  0.000000       38.61000            \\nREMARK 290   SMTRY1  13  1.000000  0.000000  0.000000       38.61000            \\nREMARK 290   SMTRY2  13  0.000000  1.000000  0.000000       38.61000            \\nREMARK 290   SMTRY3  13  0.000000  0.000000  1.000000       38.61000            \\nREMARK 290   SMTRY1  14 -1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2  14  0.000000 -1.000000  0.000000       38.61000            \\nREMARK 290   SMTRY3  14  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY1  15 -1.000000  0.000000  0.000000       38.61000            \\nREMARK 290   SMTRY2  15  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3  15  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290   SMTRY1  16  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2  16  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3  16  0.000000  0.000000 -1.000000       38.61000            \\nREMARK 290   SMTRY1  17  0.000000  0.000000  1.000000       38.61000            \\nREMARK 290   SMTRY2  17  1.000000  0.000000  0.000000       38.61000            \\nREMARK 290   SMTRY3  17  0.000000  1.000000  0.000000       38.61000            \\nREMARK 290   SMTRY1  18  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY2  18 -1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3  18  0.000000 -1.000000  0.000000       38.61000            \\nREMARK 290   SMTRY1  19  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290   SMTRY2  19 -1.000000  0.000000  0.000000       38.61000            \\nREMARK 290   SMTRY3  19  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY1  20  0.000000  0.000000 -1.000000       38.61000            \\nREMARK 290   SMTRY2  20  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3  20  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY1  21  0.000000  1.000000  0.000000       38.61000            \\nREMARK 290   SMTRY2  21  0.000000  0.000000  1.000000       38.61000            \\nREMARK 290   SMTRY3  21  1.000000  0.000000  0.000000       38.61000            \\nREMARK 290   SMTRY1  22  0.000000 -1.000000  0.000000       38.61000            \\nREMARK 290   SMTRY2  22  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY3  22 -1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY1  23  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2  23  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290   SMTRY3  23 -1.000000  0.000000  0.000000       38.61000            \\nREMARK 290   SMTRY1  24  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2  24  0.000000  0.000000 -1.000000       38.61000            \\nREMARK 290   SMTRY3  24  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290                                                                      \\nREMARK 290 REMARK: NULL                                                         \\nREMARK 300                                                                      \\nREMARK 300 BIOMOLECULE: 1                                                       \\nREMARK 300 SEE REMARK 350 FOR THE AUTHOR PROVIDED AND/OR PROGRAM                \\nREMARK 300 GENERATED ASSEMBLY INFORMATION FOR THE STRUCTURE IN                  \\nREMARK 300 THIS ENTRY. THE REMARK MAY ALSO PROVIDE INFORMATION ON               \\nREMARK 300 BURIED SURFACE AREA.                                                 \\nREMARK 350                                                                      \\nREMARK 350 COORDINATES FOR A COMPLETE MULTIMER REPRESENTING THE KNOWN           \\nREMARK 350 BIOLOGICALLY SIGNIFICANT OLIGOMERIZATION STATE OF THE                \\nREMARK 350 MOLECULE CAN BE GENERATED BY APPLYING BIOMT TRANSFORMATIONS          \\nREMARK 350 GIVEN BELOW.  BOTH NON-CRYSTALLOGRAPHIC AND                          \\nREMARK 350 CRYSTALLOGRAPHIC OPERATIONS ARE GIVEN.                               \\nREMARK 350                                                                      \\nREMARK 350 BIOMOLECULE: 1                                                       \\nREMARK 350 AUTHOR DETERMINED BIOLOGICAL UNIT: DIMERIC                           \\nREMARK 350 SOFTWARE DETERMINED QUATERNARY STRUCTURE: DIMERIC                    \\nREMARK 350 SOFTWARE USED: PISA                                                  \\nREMARK 350 TOTAL BURIED SURFACE AREA: 1520 ANGSTROM**2                          \\nREMARK 350 SURFACE AREA OF THE COMPLEX: 3380 ANGSTROM**2                        \\nREMARK 350 CHANGE IN SOLVENT FREE ENERGY: -14.0 KCAL/MOL                        \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: A, B                                  \\nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 900                                                                      \\nREMARK 900 RELATED ENTRIES                                                      \\nREMARK 900 RELATED ID: 3I3Z   RELATED DB: PDB                                   \\nREMARK 999                                                                      \\nREMARK 999 SEQUENCE                                                             \\nREMARK 999 THE CONFLICT OF THE SEQUENCE MAY DUE TO ARTIFACT.                    \\nDBREF  3I40 A    1    21  UNP    P01308   INS_HUMAN       90    110             \\nDBREF  3I40 B    1    30  UNP    P01308   INS_HUMAN       25     54             \\nSEQADV 3I40 ALA B   30  UNP  P01308    THR    54 SEE REMARK 999                 \\nSEQRES   1 A   21  GLY ILE VAL GLU GLN CYS CYS THR SER ILE CYS SER LEU          \\nSEQRES   2 A   21  TYR GLN LEU GLU ASN TYR CYS ASN                              \\nSEQRES   1 B   30  PHE VAL ASN GLN HIS LEU CYS GLY SER HIS LEU VAL GLU          \\nSEQRES   2 B   30  ALA LEU TYR LEU VAL CYS GLY GLU ARG GLY PHE PHE TYR          \\nSEQRES   3 B   30  THR PRO LYS ALA                                              \\nFORMUL   3  HOH   *35(H2 O)                                                     \\nHELIX    1   1 GLY A    1  CYS A    7  1                                   7    \\nHELIX    2   2 SER A   12  ASN A   18  1                                   7    \\nHELIX    3   3 GLY B    8  GLY B   20  1                                  13    \\nHELIX    4   4 GLU B   21  GLY B   23  5                                   3    \\nSSBOND   1 CYS A    6    CYS A   11                          1555   1555  2.03  \\nSSBOND   2 CYS A    7    CYS B    7                          1555   1555  2.04  \\nSSBOND   3 CYS A   20    CYS B   19                          1555   1555  2.04  \\nCRYST1   77.220   77.220   77.220  90.00  90.00  90.00 I 21 3       24          \\nORIGX1      1.000000  0.000000  0.000000        0.00000                         \\nORIGX2      0.000000  1.000000  0.000000        0.00000                         \\nORIGX3      0.000000  0.000000  1.000000        0.00000                         \\nSCALE1      0.012950  0.000000  0.000000        0.00000                         \\nSCALE2      0.000000  0.012950  0.000000        0.00000                         \\nSCALE3      0.000000  0.000000  0.012950        0.00000                         \\nATOM      1  N   GLY A   1     -27.279   6.238 -12.314  1.00 45.01           N  \\nATOM      2  CA  GLY A   1     -26.249   6.028 -11.313  1.00 43.47           C  \\nATOM      3  C   GLY A   1     -25.582   4.677 -11.471  1.00 34.37           C  \\nATOM      4  O   GLY A   1     -25.731   4.023 -12.501  1.00 30.09           O  \\nATOM      5  N   ILE A   2     -24.853   4.248 -10.446  1.00 32.71           N  \\nATOM      6  CA  ILE A   2     -24.070   3.023 -10.550  1.00 22.60           C  \\nATOM      7  C   ILE A   2     -24.915   1.786 -10.840  1.00 23.32           C  \\nATOM      8  O   ILE A   2     -24.469   0.865 -11.529  1.00 26.36           O  \\nATOM      9  CB  ILE A   2     -23.208   2.791  -9.302  1.00 28.24           C  \\nATOM     10  CG1 ILE A   2     -22.249   1.620  -9.538  1.00 25.03           C  \\nATOM     11  CG2 ILE A   2     -24.090   2.579  -8.068  1.00 32.70           C  \\nATOM     12  CD1 ILE A   2     -21.209   1.479  -8.457  1.00 30.60           C  \\nATOM     13  N   VAL A   3     -26.137   1.755 -10.324  1.00 27.13           N  \\nATOM     14  CA  VAL A   3     -26.993   0.599 -10.563  1.00 30.10           C  \\nATOM     15  C   VAL A   3     -27.324   0.471 -12.049  1.00 32.71           C  \\nATOM     16  O   VAL A   3     -27.258  -0.617 -12.621  1.00 28.83           O  \\nATOM     17  CB  VAL A   3     -28.284   0.657  -9.740  1.00 30.51           C  \\nATOM     18  CG1 VAL A   3     -29.194  -0.497 -10.127  1.00 35.07           C  \\nATOM     19  CG2 VAL A   3     -27.956   0.613  -8.242  1.00 27.53           C  \\nATOM     20  N   GLU A   4     -27.669   1.596 -12.667  1.00 34.97           N  \\nATOM     21  CA  GLU A   4     -28.001   1.627 -14.086  1.00 31.24           C  \\nATOM     22  C   GLU A   4     -26.797   1.224 -14.927  1.00 32.52           C  \\nATOM     23  O   GLU A   4     -26.933   0.517 -15.924  1.00 31.47           O  \\nATOM     24  CB  GLU A   4     -28.474   3.027 -14.485  1.00 38.19           C  \\nATOM     25  CG  GLU A   4     -29.829   3.420 -13.908  1.00 61.51           C  \\nATOM     26  CD  GLU A   4     -29.832   3.535 -12.387  1.00 66.16           C  \\nATOM     27  OE1 GLU A   4     -28.868   4.094 -11.814  1.00 42.60           O  \\nATOM     28  OE2 GLU A   4     -30.812   3.071 -11.762  1.00 67.86           O  \\nATOM     29  N   GLN A   5     -25.615   1.664 -14.508  1.00 28.29           N  \\nATOM     30  CA  GLN A   5     -24.393   1.410 -15.266  1.00 31.01           C  \\nATOM     31  C   GLN A   5     -23.776   0.028 -15.056  1.00 39.69           C  \\nATOM     32  O   GLN A   5     -23.149  -0.511 -15.967  1.00 26.39           O  \\nATOM     33  CB  GLN A   5     -23.349   2.492 -14.974  1.00 25.23           C  \\nATOM     34  CG  GLN A   5     -23.818   3.887 -15.346  1.00 38.13           C  \\nATOM     35  CD  GLN A   5     -24.535   3.921 -16.693  1.00 53.62           C  \\nATOM     36  OE1 GLN A   5     -24.074   3.334 -17.674  1.00 38.85           O  \\nATOM     37  NE2 GLN A   5     -25.671   4.609 -16.741  1.00 42.57           N  \\nATOM     38  N   CYS A   6     -23.940  -0.551 -13.868  1.00 22.46           N  \\nATOM     39  CA  CYS A   6     -23.214  -1.776 -13.549  1.00 20.37           C  \\nATOM     40  C   CYS A   6     -24.114  -2.963 -13.225  1.00 18.21           C  \\nATOM     41  O   CYS A   6     -23.677  -4.107 -13.293  1.00 21.83           O  \\nATOM     42  CB  CYS A   6     -22.235  -1.524 -12.396  1.00 22.32           C  \\nATOM     43  SG  CYS A   6     -20.756  -0.602 -12.893  1.00 27.82           S  \\nATOM     44  N   CYS A   7     -25.359  -2.685 -12.860  1.00 22.16           N  \\nATOM     45  CA  CYS A   7     -26.307  -3.744 -12.520  1.00 21.25           C  \\nATOM     46  C   CYS A   7     -27.245  -4.032 -13.690  1.00 31.44           C  \\nATOM     47  O   CYS A   7     -27.307  -5.155 -14.190  1.00 24.09           O  \\nATOM     48  CB  CYS A   7     -27.119  -3.351 -11.286  1.00 22.97           C  \\nATOM     49  SG  CYS A   7     -28.355  -4.584 -10.791  1.00 28.27           S  \\nATOM     50  N   THR A   8     -27.967  -3.001 -14.122  1.00 29.15           N  \\nATOM     51  CA  THR A   8     -28.901  -3.121 -15.236  1.00 29.59           C  \\nATOM     52  C   THR A   8     -28.147  -3.334 -16.542  1.00 34.52           C  \\nATOM     53  O   THR A   8     -28.647  -3.986 -17.454  1.00 38.61           O  \\nATOM     54  CB  THR A   8     -29.780  -1.868 -15.371  1.00 31.24           C  \\nATOM     55  OG1 THR A   8     -30.023  -1.303 -14.080  1.00 41.12           O  \\nATOM     56  CG2 THR A   8     -31.100  -2.218 -16.029  1.00 51.07           C  \\nATOM     57  N   SER A   9     -26.948  -2.769 -16.630  1.00 26.15           N  \\nATOM     58  CA  SER A   9     -26.064  -3.019 -17.765  1.00 21.41           C  \\nATOM     59  C   SER A   9     -24.699  -3.529 -17.309  1.00 32.36           C  \\nATOM     60  O   SER A   9     -24.419  -3.594 -16.111  1.00 29.40           O  \\nATOM     61  CB  SER A   9     -25.910  -1.765 -18.626  1.00 32.53           C  \\nATOM     62  OG  SER A   9     -25.465  -0.670 -17.851  1.00 47.07           O  \\nATOM     63  N   ILE A  10     -23.849  -3.891 -18.261  1.00 24.43           N  \\nATOM     64  CA  ILE A  10     -22.562  -4.492 -17.924  1.00 24.22           C  \\nATOM     65  C   ILE A  10     -21.591  -3.454 -17.367  1.00 30.49           C  \\nATOM     66  O   ILE A  10     -21.420  -2.380 -17.934  1.00 26.28           O  \\nATOM     67  CB  ILE A  10     -21.942  -5.216 -19.128  1.00 26.99           C  \\nATOM     68  CG1 ILE A  10     -22.880  -6.328 -19.595  1.00 26.49           C  \\nATOM     69  CG2 ILE A  10     -20.577  -5.790 -18.770  1.00 25.10           C  \\nATOM     70  CD1 ILE A  10     -22.302  -7.179 -20.689  1.00 44.31           C  \\nATOM     71  N   CYS A  11     -20.972  -3.790 -16.240  1.00 28.14           N  \\nATOM     72  CA  CYS A  11     -20.088  -2.876 -15.534  1.00 25.92           C  \\nATOM     73  C   CYS A  11     -18.752  -2.779 -16.258  1.00 36.06           C  \\nATOM     74  O   CYS A  11     -18.491  -3.524 -17.204  1.00 27.25           O  \\nATOM     75  CB  CYS A  11     -19.867  -3.387 -14.106  1.00 23.76           C  \\nATOM     76  SG  CYS A  11     -19.406  -2.114 -12.908  1.00 28.09           S  \\nATOM     77  N   SER A  12     -17.902  -1.865 -15.807  1.00 25.78           N  \\nATOM     78  CA  SER A  12     -16.554  -1.767 -16.349  1.00 22.91           C  \\nATOM     79  C   SER A  12     -15.634  -1.220 -15.281  1.00 19.20           C  \\nATOM     80  O   SER A  12     -16.076  -0.508 -14.375  1.00 18.62           O  \\nATOM     81  CB  SER A  12     -16.530  -0.845 -17.559  1.00 24.50           C  \\nATOM     82  OG  SER A  12     -16.681   0.502 -17.157  1.00 24.75           O  \\nATOM     83  N   LEU A  13     -14.346  -1.528 -15.392  1.00 22.17           N  \\nATOM     84  CA  LEU A  13     -13.378  -1.013 -14.426  1.00 19.72           C  \\nATOM     85  C   LEU A  13     -13.412   0.508 -14.438  1.00 22.22           C  \\nATOM     86  O   LEU A  13     -13.329   1.158 -13.393  1.00 17.29           O  \\nATOM     87  CB  LEU A  13     -11.971  -1.517 -14.751  1.00 19.22           C  \\nATOM     88  CG  LEU A  13     -10.860  -0.998 -13.840  1.00 17.99           C  \\nATOM     89  CD1 LEU A  13     -11.104  -1.386 -12.388  1.00 18.06           C  \\nATOM     90  CD2 LEU A  13      -9.522  -1.530 -14.308  1.00 15.25           C  \\nATOM     91  N   TYR A  14     -13.550   1.075 -15.632  1.00 19.94           N  \\nATOM     92  CA  TYR A  14     -13.559   2.521 -15.774  1.00 22.72           C  \\nATOM     93  C   TYR A  14     -14.638   3.140 -14.893  1.00 22.64           C  \\nATOM     94  O   TYR A  14     -14.391   4.094 -14.152  1.00 25.33           O  \\nATOM     95  CB ATYR A  14     -13.771   2.911 -17.239  0.50 27.81           C  \\nATOM     96  CB BTYR A  14     -13.786   2.928 -17.233  0.50 27.85           C  \\nATOM     97  CG ATYR A  14     -13.608   4.388 -17.494  0.50 28.31           C  \\nATOM     98  CG BTYR A  14     -13.829   4.427 -17.427  0.50 28.20           C  \\nATOM     99  CD1ATYR A  14     -12.442   4.893 -18.059  0.50 23.25           C  \\nATOM    100  CD1BTYR A  14     -12.714   5.121 -17.879  0.50 25.75           C  \\nATOM    101  CD2ATYR A  14     -14.614   5.282 -17.153  0.50 28.36           C  \\nATOM    102  CD2BTYR A  14     -14.980   5.152 -17.138  0.50 27.68           C  \\nATOM    103  CE1ATYR A  14     -12.289   6.252 -18.283  0.50 27.32           C  \\nATOM    104  CE1BTYR A  14     -12.746   6.496 -18.047  0.50 27.49           C  \\nATOM    105  CE2ATYR A  14     -14.472   6.636 -17.372  0.50 29.32           C  \\nATOM    106  CE2BTYR A  14     -15.021   6.527 -17.300  0.50 28.26           C  \\nATOM    107  CZ ATYR A  14     -13.311   7.117 -17.938  0.50 29.06           C  \\nATOM    108  CZ BTYR A  14     -13.902   7.193 -17.757  0.50 29.57           C  \\nATOM    109  OH ATYR A  14     -13.179   8.469 -18.153  0.50 33.01           O  \\nATOM    110  OH BTYR A  14     -13.938   8.561 -17.922  0.50 32.79           O  \\nATOM    111  N   GLN A  15     -15.837   2.584 -14.969  1.00 21.27           N  \\nATOM    112  CA  GLN A  15     -16.962   3.129 -14.222  1.00 21.51           C  \\nATOM    113  C   GLN A  15     -16.837   2.882 -12.722  1.00 17.30           C  \\nATOM    114  O   GLN A  15     -17.121   3.764 -11.917  1.00 20.82           O  \\nATOM    115  CB  GLN A  15     -18.275   2.561 -14.758  1.00 25.25           C  \\nATOM    116  CG  GLN A  15     -18.527   2.905 -16.214  1.00 32.75           C  \\nATOM    117  CD  GLN A  15     -19.997   3.039 -16.527  1.00 55.05           C  \\nATOM    118  OE1 GLN A  15     -20.680   3.914 -15.987  1.00 53.01           O  \\nATOM    119  NE2 GLN A  15     -20.497   2.177 -17.410  1.00 50.66           N  \\nATOM    120  N   LEU A  16     -16.399   1.687 -12.346  1.00 15.37           N  \\nATOM    121  CA  LEU A  16     -16.236   1.380 -10.930  1.00 17.02           C  \\nATOM    122  C   LEU A  16     -15.301   2.382 -10.273  1.00 16.55           C  \\nATOM    123  O   LEU A  16     -15.565   2.868  -9.172  1.00 19.70           O  \\nATOM    124  CB  LEU A  16     -15.703  -0.037 -10.742  1.00 24.64           C  \\nATOM    125  CG  LEU A  16     -16.709  -1.151 -11.013  1.00 24.23           C  \\nATOM    126  CD1 LEU A  16     -16.007  -2.496 -11.058  1.00 21.90           C  \\nATOM    127  CD2 LEU A  16     -17.786  -1.124  -9.936  1.00 18.48           C  \\nATOM    128  N   GLU A  17     -14.199   2.703 -10.950  1.00 17.86           N  \\nATOM    129  CA  GLU A  17     -13.219   3.608 -10.365  1.00 19.38           C  \\nATOM    130  C   GLU A  17     -13.763   5.026 -10.180  1.00 18.73           C  \\nATOM    131  O   GLU A  17     -13.259   5.782  -9.351  1.00 22.43           O  \\nATOM    132  CB  GLU A  17     -11.910   3.598 -11.155  1.00 20.40           C  \\nATOM    133  CG  GLU A  17     -11.090   2.343 -10.898  1.00 20.17           C  \\nATOM    134  CD  GLU A  17      -9.623   2.514 -11.240  1.00 22.14           C  \\nATOM    135  OE1 GLU A  17      -9.301   3.347 -12.110  1.00 22.42           O  \\nATOM    136  OE2 GLU A  17      -8.792   1.805 -10.640  1.00 27.73           O  \\nATOM    137  N   ASN A  18     -14.799   5.374 -10.932  1.00 20.29           N  \\nATOM    138  CA  ASN A  18     -15.480   6.657 -10.735  1.00 20.00           C  \\nATOM    139  C   ASN A  18     -16.080   6.766  -9.334  1.00 26.36           C  \\nATOM    140  O   ASN A  18     -16.469   7.849  -8.890  1.00 28.46           O  \\nATOM    141  CB  ASN A  18     -16.587   6.839 -11.774  1.00 21.66           C  \\nATOM    142  CG  ASN A  18     -16.061   7.344 -13.113  1.00 25.89           C  \\nATOM    143  OD1 ASN A  18     -15.007   7.974 -13.180  1.00 32.32           O  \\nATOM    144  ND2 ASN A  18     -16.802   7.070 -14.180  1.00 38.57           N  \\nATOM    145  N   TYR A  19     -16.163   5.640  -8.638  1.00 20.84           N  \\nATOM    146  CA  TYR A  19     -16.803   5.618  -7.327  1.00 18.56           C  \\nATOM    147  C   TYR A  19     -15.822   5.394  -6.180  1.00 23.20           C  \\nATOM    148  O   TYR A  19     -16.223   5.337  -5.018  1.00 27.54           O  \\nATOM    149  CB  TYR A  19     -17.945   4.599  -7.298  1.00 20.89           C  \\nATOM    150  CG  TYR A  19     -19.068   4.966  -8.236  1.00 21.62           C  \\nATOM    151  CD1 TYR A  19     -19.115   4.453  -9.521  1.00 23.43           C  \\nATOM    152  CD2 TYR A  19     -20.066   5.848  -7.844  1.00 26.68           C  \\nATOM    153  CE1 TYR A  19     -20.126   4.796 -10.385  1.00 24.39           C  \\nATOM    154  CE2 TYR A  19     -21.082   6.200  -8.708  1.00 31.02           C  \\nATOM    155  CZ  TYR A  19     -21.104   5.672  -9.979  1.00 28.36           C  \\nATOM    156  OH  TYR A  19     -22.114   6.009 -10.853  1.00 42.08           O  \\nATOM    157  N   CYS A  20     -14.534   5.281  -6.499  1.00 21.90           N  \\nATOM    158  CA  CYS A  20     -13.510   5.251  -5.459  1.00 19.66           C  \\nATOM    159  C   CYS A  20     -13.334   6.645  -4.882  1.00 28.69           C  \\nATOM    160  O   CYS A  20     -13.526   7.639  -5.585  1.00 30.53           O  \\nATOM    161  CB  CYS A  20     -12.169   4.783  -6.021  1.00 22.84           C  \\nATOM    162  SG  CYS A  20     -12.209   3.187  -6.829  1.00 22.45           S  \\nATOM    163  N   ASN A  21     -12.955   6.716  -3.609  1.00 26.51           N  \\nATOM    164  CA  ASN A  21     -12.630   7.993  -2.976  1.00 36.60           C  \\nATOM    165  C   ASN A  21     -11.279   8.511  -3.448  1.00 35.58           C  \\nATOM    166  O   ASN A  21     -10.515   7.790  -4.091  1.00 36.84           O  \\nATOM    167  CB  ASN A  21     -12.615   7.863  -1.450  1.00 34.89           C  \\nATOM    168  CG  ASN A  21     -13.978   7.548  -0.878  1.00 33.46           C  \\nATOM    169  OD1 ASN A  21     -14.107   6.726   0.029  1.00 48.60           O  \\nATOM    170  ND2 ASN A  21     -15.008   8.198  -1.408  1.00 39.53           N  \\nATOM    171  OXT ASN A  21     -10.922   9.659  -3.189  1.00 58.00           O  \\nTER     172      ASN A  21                                                      \\nATOM    173  N   PHE B   1     -13.174  -9.740 -13.251  1.00 44.56           N  \\nATOM    174  CA  PHE B   1     -14.387  -9.892 -12.452  1.00 53.84           C  \\nATOM    175  C   PHE B   1     -15.310  -8.684 -12.590  1.00 44.51           C  \\nATOM    176  O   PHE B   1     -16.451  -8.715 -12.133  1.00 40.17           O  \\nATOM    177  CB  PHE B   1     -14.042 -10.126 -10.973  1.00 47.36           C  \\nATOM    178  CG  PHE B   1     -13.222  -9.024 -10.356  1.00 43.05           C  \\nATOM    179  CD1 PHE B   1     -13.816  -8.050  -9.571  1.00 39.13           C  \\nATOM    180  CD2 PHE B   1     -11.854  -8.964 -10.565  1.00 48.11           C  \\nATOM    181  CE1 PHE B   1     -13.059  -7.037  -9.008  1.00 29.34           C  \\nATOM    182  CE2 PHE B   1     -11.093  -7.959 -10.002  1.00 34.97           C  \\nATOM    183  CZ  PHE B   1     -11.696  -6.993  -9.224  1.00 36.24           C  \\nATOM    184  N   VAL B   2     -14.819  -7.623 -13.228  1.00 27.44           N  \\nATOM    185  CA  VAL B   2     -15.568  -6.367 -13.273  1.00 25.65           C  \\nATOM    186  C   VAL B   2     -16.402  -6.197 -14.541  1.00 30.55           C  \\nATOM    187  O   VAL B   2     -17.249  -5.306 -14.628  1.00 30.98           O  \\nATOM    188  CB  VAL B   2     -14.633  -5.163 -13.114  1.00 32.50           C  \\nATOM    189  CG1 VAL B   2     -14.056  -5.139 -11.705  1.00 29.74           C  \\nATOM    190  CG2 VAL B   2     -13.519  -5.227 -14.141  1.00 45.24           C  \\nATOM    191  N   ASN B   3     -16.158  -7.045 -15.528  1.00 28.57           N  \\nATOM    192  CA  ASN B   3     -16.844  -6.904 -16.806  1.00 31.00           C  \\nATOM    193  C   ASN B   3     -18.072  -7.800 -16.928  1.00 29.49           C  \\nATOM    194  O   ASN B   3     -18.115  -8.720 -17.743  1.00 28.57           O  \\nATOM    195  CB  ASN B   3     -15.854  -7.122 -17.946  1.00 31.67           C  \\nATOM    196  CG  ASN B   3     -14.703  -6.139 -17.888  1.00 35.45           C  \\nATOM    197  OD1 ASN B   3     -13.561  -6.507 -17.600  1.00 38.37           O  \\nATOM    198  ND2 ASN B   3     -15.009  -4.871 -18.113  1.00 23.34           N  \\nATOM    199  N   GLN B   4     -19.073  -7.514 -16.099  1.00 31.90           N  \\nATOM    200  CA  GLN B   4     -20.313  -8.287 -16.082  1.00 28.96           C  \\nATOM    201  C   GLN B   4     -21.397  -7.521 -15.328  1.00 23.12           C  \\nATOM    202  O   GLN B   4     -21.154  -6.432 -14.812  1.00 26.67           O  \\nATOM    203  CB  GLN B   4     -20.079  -9.648 -15.425  1.00 33.50           C  \\nATOM    204  CG  GLN B   4     -19.608  -9.556 -13.987  1.00 31.03           C  \\nATOM    205  CD  GLN B   4     -19.344 -10.918 -13.379  1.00 54.72           C  \\nATOM    206  OE1 GLN B   4     -20.210 -11.794 -13.390  1.00 51.79           O  \\nATOM    207  NE2 GLN B   4     -18.144 -11.105 -12.840  1.00 41.70           N  \\nATOM    208  N   HIS B   5     -22.599  -8.080 -15.281  1.00 25.17           N  \\nATOM    209  CA  HIS B   5     -23.666  -7.471 -14.503  1.00 26.87           C  \\nATOM    210  C   HIS B   5     -23.358  -7.685 -13.035  1.00 26.33           C  \\nATOM    211  O   HIS B   5     -23.193  -8.822 -12.595  1.00 25.34           O  \\nATOM    212  CB  HIS B   5     -25.013  -8.109 -14.834  1.00 24.65           C  \\nATOM    213  CG  HIS B   5     -25.462  -7.871 -16.239  1.00 27.43           C  \\nATOM    214  ND1 HIS B   5     -26.235  -6.787 -16.598  1.00 29.87           N  \\nATOM    215  CD2 HIS B   5     -25.241  -8.570 -17.377  1.00 27.84           C  \\nATOM    216  CE1 HIS B   5     -26.478  -6.833 -17.896  1.00 24.52           C  \\nATOM    217  NE2 HIS B   5     -25.885  -7.905 -18.393  1.00 35.03           N  \\nATOM    218  N   LEU B   6     -23.267  -6.592 -12.287  1.00 22.69           N  \\nATOM    219  CA  LEU B   6     -22.974  -6.660 -10.862  1.00 21.06           C  \\nATOM    220  C   LEU B   6     -24.052  -5.932 -10.079  1.00 21.50           C  \\nATOM    221  O   LEU B   6     -24.197  -4.715 -10.194  1.00 22.66           O  \\nATOM    222  CB  LEU B   6     -21.612  -6.034 -10.568  1.00 20.56           C  \\nATOM    223  CG  LEU B   6     -20.418  -6.665 -11.281  1.00 20.16           C  \\nATOM    224  CD1 LEU B   6     -19.190  -5.786 -11.103  1.00 20.54           C  \\nATOM    225  CD2 LEU B   6     -20.155  -8.064 -10.763  1.00 25.22           C  \\nATOM    226  N   CYS B   7     -24.800  -6.679  -9.273  1.00 20.58           N  \\nATOM    227  CA  CYS B   7     -25.899  -6.104  -8.513  1.00 22.46           C  \\nATOM    228  C   CYS B   7     -25.754  -6.421  -7.034  1.00 22.72           C  \\nATOM    229  O   CYS B   7     -25.231  -7.471  -6.665  1.00 21.90           O  \\nATOM    230  CB  CYS B   7     -27.236  -6.664  -9.009  1.00 28.16           C  \\nATOM    231  SG  CYS B   7     -27.554  -6.459 -10.788  1.00 29.45           S  \\nATOM    232  N   GLY B   8     -26.229  -5.514  -6.191  1.00 21.36           N  \\nATOM    233  CA  GLY B   8     -26.233  -5.745  -4.759  1.00 22.98           C  \\nATOM    234  C   GLY B   8     -24.859  -6.051  -4.209  1.00 20.91           C  \\nATOM    235  O   GLY B   8     -23.911  -5.316  -4.465  1.00 23.25           O  \\nATOM    236  N   SER B   9     -24.741  -7.136  -3.447  1.00 18.12           N  \\nATOM    237  CA  SER B   9     -23.484  -7.444  -2.778  1.00 15.90           C  \\nATOM    238  C   SER B   9     -22.360  -7.702  -3.780  1.00 18.45           C  \\nATOM    239  O   SER B   9     -21.183  -7.509  -3.473  1.00 17.21           O  \\nATOM    240  CB  SER B   9     -23.648  -8.663  -1.863  1.00 19.34           C  \\nATOM    241  OG  SER B   9     -23.919  -9.813  -2.638  1.00 20.35           O  \\nATOM    242  N   HIS B  10     -22.730  -8.151  -4.972  1.00 14.91           N  \\nATOM    243  CA  HIS B  10     -21.764  -8.403  -6.039  1.00 17.86           C  \\nATOM    244  C   HIS B  10     -21.093  -7.097  -6.470  1.00 18.59           C  \\nATOM    245  O   HIS B  10     -19.891  -7.055  -6.725  1.00 17.96           O  \\nATOM    246  CB  HIS B  10     -22.471  -9.047  -7.236  1.00 18.56           C  \\nATOM    247  CG  HIS B  10     -23.081 -10.384  -6.933  1.00 23.08           C  \\nATOM    248  ND1 HIS B  10     -23.461 -11.269  -7.919  1.00 27.72           N  \\nATOM    249  CD2 HIS B  10     -23.366 -10.990  -5.755  1.00 23.69           C  \\nATOM    250  CE1 HIS B  10     -23.951 -12.364  -7.363  1.00 29.56           C  \\nATOM    251  NE2 HIS B  10     -23.906 -12.220  -6.050  1.00 22.03           N  \\nATOM    252  N   LEU B  11     -21.886  -6.037  -6.550  1.00 16.34           N  \\nATOM    253  CA  LEU B  11     -21.398  -4.713  -6.938  1.00 16.50           C  \\nATOM    254  C   LEU B  11     -20.523  -4.120  -5.847  1.00 19.44           C  \\nATOM    255  O   LEU B  11     -19.446  -3.585  -6.105  1.00 17.25           O  \\nATOM    256  CB  LEU B  11     -22.588  -3.788  -7.211  1.00 16.97           C  \\nATOM    257  CG  LEU B  11     -22.268  -2.390  -7.735  1.00 22.79           C  \\nATOM    258  CD1 LEU B  11     -21.211  -2.486  -8.812  1.00 17.51           C  \\nATOM    259  CD2 LEU B  11     -23.529  -1.754  -8.277  1.00 22.41           C  \\nATOM    260  N   VAL B  12     -20.990  -4.218  -4.610  1.00 15.40           N  \\nATOM    261  CA  VAL B  12     -20.242  -3.690  -3.490  1.00 13.81           C  \\nATOM    262  C   VAL B  12     -18.902  -4.422  -3.332  1.00 15.23           C  \\nATOM    263  O   VAL B  12     -17.879  -3.825  -2.988  1.00 14.98           O  \\nATOM    264  CB  VAL B  12     -21.096  -3.776  -2.212  1.00 24.43           C  \\nATOM    265  CG1 VAL B  12     -20.252  -3.609  -1.029  1.00 30.29           C  \\nATOM    266  CG2 VAL B  12     -22.169  -2.694  -2.241  1.00 21.29           C  \\nATOM    267  N   GLU B  13     -18.912  -5.721  -3.589  1.00 14.50           N  \\nATOM    268  CA  GLU B  13     -17.695  -6.515  -3.515  1.00 14.19           C  \\nATOM    269  C   GLU B  13     -16.682  -6.056  -4.574  1.00 15.30           C  \\nATOM    270  O   GLU B  13     -15.481  -5.936  -4.304  1.00 16.00           O  \\nATOM    271  CB  GLU B  13     -18.046  -7.986  -3.712  1.00 22.91           C  \\nATOM    272  CG  GLU B  13     -16.884  -8.875  -4.073  1.00 31.68           C  \\nATOM    273  CD  GLU B  13     -17.296 -10.336  -4.139  1.00 35.29           C  \\nATOM    274  OE1 GLU B  13     -17.726 -10.873  -3.092  1.00 24.65           O  \\nATOM    275  OE2 GLU B  13     -17.204 -10.932  -5.236  1.00 35.79           O  \\nATOM    276  N   ALA B  14     -17.166  -5.811  -5.782  1.00 16.94           N  \\nATOM    277  CA  ALA B  14     -16.298  -5.308  -6.856  1.00 19.02           C  \\nATOM    278  C   ALA B  14     -15.701  -3.941  -6.523  1.00 17.95           C  \\nATOM    279  O   ALA B  14     -14.507  -3.704  -6.731  1.00 16.40           O  \\nATOM    280  CB  ALA B  14     -17.065  -5.240  -8.157  1.00 19.13           C  \\nATOM    281  N   LEU B  15     -16.536  -3.036  -6.021  1.00 15.53           N  \\nATOM    282  CA  LEU B  15     -16.066  -1.724  -5.592  1.00 13.58           C  \\nATOM    283  C   LEU B  15     -14.977  -1.846  -4.540  1.00 20.33           C  \\nATOM    284  O   LEU B  15     -13.957  -1.159  -4.594  1.00 17.96           O  \\nATOM    285  CB  LEU B  15     -17.228  -0.890  -5.054  1.00 17.56           C  \\nATOM    286  CG  LEU B  15     -18.132  -0.253  -6.108  1.00 19.95           C  \\nATOM    287  CD1 LEU B  15     -19.434   0.217  -5.476  1.00 19.70           C  \\nATOM    288  CD2 LEU B  15     -17.420   0.908  -6.806  1.00 19.43           C  \\nATOM    289  N   TYR B  16     -15.196  -2.733  -3.580  1.00 16.41           N  \\nATOM    290  CA  TYR B  16     -14.214  -2.982  -2.528  1.00 14.22           C  \\nATOM    291  C   TYR B  16     -12.884  -3.467  -3.093  1.00 18.76           C  \\nATOM    292  O   TYR B  16     -11.823  -2.978  -2.708  1.00 14.92           O  \\nATOM    293  CB  TYR B  16     -14.766  -4.015  -1.551  1.00 13.58           C  \\nATOM    294  CG  TYR B  16     -13.748  -4.582  -0.595  1.00 15.85           C  \\nATOM    295  CD1 TYR B  16     -13.213  -3.811   0.431  1.00 19.91           C  \\nATOM    296  CD2 TYR B  16     -13.338  -5.909  -0.707  1.00 14.31           C  \\nATOM    297  CE1 TYR B  16     -12.290  -4.347   1.318  1.00 18.93           C  \\nATOM    298  CE2 TYR B  16     -12.419  -6.445   0.163  1.00 16.89           C  \\nATOM    299  CZ  TYR B  16     -11.899  -5.665   1.173  1.00 19.03           C  \\nATOM    300  OH  TYR B  16     -10.991  -6.225   2.037  1.00 19.26           O  \\nATOM    301  N   LEU B  17     -12.941  -4.437  -4.002  1.00 17.26           N  \\nATOM    302  CA  LEU B  17     -11.725  -4.998  -4.583  1.00 17.15           C  \\nATOM    303  C   LEU B  17     -10.983  -3.979  -5.439  1.00 20.48           C  \\nATOM    304  O   LEU B  17      -9.751  -3.889  -5.407  1.00 20.37           O  \\nATOM    305  CB  LEU B  17     -12.060  -6.221  -5.428  1.00 16.44           C  \\nATOM    306  CG  LEU B  17     -12.491  -7.468  -4.651  1.00 17.44           C  \\nATOM    307  CD1 LEU B  17     -12.867  -8.590  -5.593  1.00 22.58           C  \\nATOM    308  CD2 LEU B  17     -11.383  -7.898  -3.717  1.00 22.13           C  \\nATOM    309  N   VAL B  18     -11.738  -3.226  -6.224  1.00 18.15           N  \\nATOM    310  CA  VAL B  18     -11.150  -2.270  -7.151  1.00 18.50           C  \\nATOM    311  C   VAL B  18     -10.557  -1.063  -6.431  1.00 22.53           C  \\nATOM    312  O   VAL B  18      -9.454  -0.628  -6.747  1.00 21.63           O  \\nATOM    313  CB  VAL B  18     -12.188  -1.824  -8.207  1.00 19.51           C  \\nATOM    314  CG1 VAL B  18     -11.714  -0.594  -8.955  1.00 28.07           C  \\nATOM    315  CG2 VAL B  18     -12.468  -2.973  -9.165  1.00 23.98           C  \\nATOM    316  N   CYS B  19     -11.268  -0.548  -5.433  1.00 19.45           N  \\nATOM    317  CA  CYS B  19     -10.874   0.709  -4.803  1.00 21.28           C  \\nATOM    318  C   CYS B  19      -9.819   0.588  -3.702  1.00 26.60           C  \\nATOM    319  O   CYS B  19      -9.167   1.576  -3.355  1.00 25.82           O  \\nATOM    320  CB  CYS B  19     -12.102   1.455  -4.285  1.00 20.90           C  \\nATOM    321  SG  CYS B  19     -13.231   1.932  -5.593  1.00 21.21           S  \\nATOM    322  N   GLY B  20      -9.653  -0.608  -3.150  1.00 25.20           N  \\nATOM    323  CA  GLY B  20      -8.606  -0.825  -2.167  1.00 33.63           C  \\nATOM    324  C   GLY B  20      -8.661   0.162  -1.014  1.00 39.29           C  \\nATOM    325  O   GLY B  20      -9.737   0.464  -0.501  1.00 27.87           O  \\nATOM    326  N   GLU B  21      -7.507   0.678  -0.599  1.00 30.38           N  \\nATOM    327  CA  GLU B  21      -7.474   1.534   0.591  1.00 34.13           C  \\nATOM    328  C   GLU B  21      -8.014   2.944   0.351  1.00 23.63           C  \\nATOM    329  O   GLU B  21      -8.123   3.737   1.283  1.00 35.45           O  \\nATOM    330  CB  GLU B  21      -6.075   1.569   1.220  1.00 45.91           C  \\nATOM    331  CG  GLU B  21      -5.015   2.280   0.401  1.00 57.50           C  \\nATOM    332  CD  GLU B  21      -3.631   2.179   1.034  1.00 75.22           C  \\nATOM    333  OE1 GLU B  21      -2.707   2.876   0.563  1.00 79.46           O  \\nATOM    334  OE2 GLU B  21      -3.470   1.401   2.002  1.00 57.31           O  \\nATOM    335  N   ARG B  22      -8.356   3.251  -0.897  1.00 27.41           N  \\nATOM    336  CA  ARG B  22      -9.040   4.498  -1.213  1.00 28.81           C  \\nATOM    337  C   ARG B  22     -10.456   4.461  -0.645  1.00 33.76           C  \\nATOM    338  O   ARG B  22     -11.008   5.480  -0.235  1.00 31.35           O  \\nATOM    339  CB  ARG B  22      -9.130   4.700  -2.726  1.00 30.48           C  \\nATOM    340  CG  ARG B  22      -7.876   5.228  -3.400  1.00 38.41           C  \\nATOM    341  CD  ARG B  22      -8.072   5.288  -4.922  1.00 44.34           C  \\nATOM    342  NE  ARG B  22      -8.003   3.963  -5.546  1.00 61.53           N  \\nATOM    343  CZ  ARG B  22      -8.246   3.711  -6.833  1.00 56.00           C  \\nATOM    344  NH1 ARG B  22      -8.151   2.467  -7.288  1.00 37.21           N  \\nATOM    345  NH2 ARG B  22      -8.588   4.690  -7.664  1.00 33.73           N  \\nATOM    346  N   GLY B  23     -11.048   3.276  -0.627  1.00 23.72           N  \\nATOM    347  CA  GLY B  23     -12.442   3.151  -0.243  1.00 25.22           C  \\nATOM    348  C   GLY B  23     -13.323   3.633  -1.376  1.00 26.84           C  \\nATOM    349  O   GLY B  23     -12.832   3.954  -2.460  1.00 20.71           O  \\nATOM    350  N   PHE B  24     -14.625   3.700  -1.132  1.00 18.98           N  \\nATOM    351  CA  PHE B  24     -15.558   4.055  -2.188  1.00 17.55           C  \\nATOM    352  C   PHE B  24     -16.875   4.561  -1.614  1.00 23.50           C  \\nATOM    353  O   PHE B  24     -17.092   4.530  -0.402  1.00 24.02           O  \\nATOM    354  CB  PHE B  24     -15.811   2.837  -3.086  1.00 17.36           C  \\nATOM    355  CG  PHE B  24     -16.428   1.661  -2.361  1.00 19.89           C  \\nATOM    356  CD1 PHE B  24     -15.628   0.691  -1.776  1.00 18.48           C  \\nATOM    357  CD2 PHE B  24     -17.811   1.527  -2.269  1.00 22.89           C  \\nATOM    358  CE1 PHE B  24     -16.189  -0.390  -1.107  1.00 21.90           C  \\nATOM    359  CE2 PHE B  24     -18.378   0.446  -1.597  1.00 19.48           C  \\nATOM    360  CZ  PHE B  24     -17.562  -0.513  -1.015  1.00 19.73           C  \\nATOM    361  N   PHE B  25     -17.742   5.048  -2.487  1.00 23.02           N  \\nATOM    362  CA  PHE B  25     -19.108   5.343  -2.088  1.00 26.05           C  \\nATOM    363  C   PHE B  25     -20.081   4.558  -2.948  1.00 29.87           C  \\nATOM    364  O   PHE B  25     -19.912   4.448  -4.158  1.00 25.71           O  \\nATOM    365  CB  PHE B  25     -19.410   6.848  -2.118  1.00 30.50           C  \\nATOM    366  CG  PHE B  25     -19.035   7.523  -3.399  1.00 36.46           C  \\nATOM    367  CD1 PHE B  25     -17.729   7.929  -3.625  1.00 41.85           C  \\nATOM    368  CD2 PHE B  25     -19.989   7.772  -4.374  1.00 47.50           C  \\nATOM    369  CE1 PHE B  25     -17.376   8.562  -4.808  1.00 51.84           C  \\nATOM    370  CE2 PHE B  25     -19.645   8.407  -5.560  1.00 37.96           C  \\nATOM    371  CZ  PHE B  25     -18.336   8.802  -5.778  1.00 47.77           C  \\nATOM    372  N   TYR B  26     -21.090   3.991  -2.301  1.00 26.11           N  \\nATOM    373  CA  TYR B  26     -22.087   3.184  -2.976  1.00 19.37           C  \\nATOM    374  C   TYR B  26     -23.429   3.866  -2.826  1.00 26.37           C  \\nATOM    375  O   TYR B  26     -23.966   3.954  -1.726  1.00 25.08           O  \\nATOM    376  CB  TYR B  26     -22.136   1.795  -2.350  1.00 18.99           C  \\nATOM    377  CG  TYR B  26     -23.255   0.930  -2.876  1.00 21.56           C  \\nATOM    378  CD1 TYR B  26     -24.280   0.506  -2.042  1.00 24.62           C  \\nATOM    379  CD2 TYR B  26     -23.285   0.533  -4.203  1.00 23.44           C  \\nATOM    380  CE1 TYR B  26     -25.304  -0.295  -2.516  1.00 26.30           C  \\nATOM    381  CE2 TYR B  26     -24.304  -0.265  -4.690  1.00 25.09           C  \\nATOM    382  CZ  TYR B  26     -25.313  -0.675  -3.842  1.00 27.86           C  \\nATOM    383  OH  TYR B  26     -26.328  -1.469  -4.320  1.00 29.25           O  \\nATOM    384  N   THR B  27     -23.960   4.365  -3.936  1.00 28.14           N  \\nATOM    385  CA  THR B  27     -25.185   5.147  -3.909  1.00 37.14           C  \\nATOM    386  C   THR B  27     -26.169   4.594  -4.936  1.00 32.53           C  \\nATOM    387  O   THR B  27     -26.193   5.035  -6.082  1.00 36.20           O  \\nATOM    388  CB  THR B  27     -24.889   6.637  -4.188  1.00 36.89           C  \\nATOM    389  OG1 THR B  27     -24.271   6.772  -5.473  1.00 57.47           O  \\nATOM    390  CG2 THR B  27     -23.936   7.182  -3.151  1.00 33.01           C  \\nATOM    391  N   PRO B  28     -26.989   3.620  -4.517  1.00 29.25           N  \\nATOM    392  CA  PRO B  28     -27.893   2.887  -5.411  1.00 32.94           C  \\nATOM    393  C   PRO B  28     -29.081   3.726  -5.872  1.00 32.76           C  \\nATOM    394  O   PRO B  28     -29.671   3.417  -6.908  1.00 36.05           O  \\nATOM    395  CB  PRO B  28     -28.401   1.730  -4.542  1.00 31.48           C  \\nATOM    396  CG  PRO B  28     -27.658   1.813  -3.247  1.00 28.54           C  \\nATOM    397  CD  PRO B  28     -27.128   3.190  -3.118  1.00 34.39           C  \\nATOM    398  N   LYS B  29     -29.436   4.751  -5.105  1.00 34.79           N  \\nATOM    399  CA  LYS B  29     -30.532   5.642  -5.483  1.00 49.07           C  \\nATOM    400  C   LYS B  29     -30.051   6.834  -6.309  1.00 63.70           C  \\nATOM    401  O   LYS B  29     -30.669   7.191  -7.315  1.00 73.47           O  \\nATOM    402  CB  LYS B  29     -31.305   6.122  -4.251  1.00 62.86           C  \\nATOM    403  CG  LYS B  29     -32.706   5.526  -4.105  1.00 74.15           C  \\nATOM    404  CD  LYS B  29     -32.690   4.118  -3.525  1.00 72.32           C  \\nATOM    405  CE  LYS B  29     -32.356   3.061  -4.572  1.00 68.55           C  \\nATOM    406  NZ  LYS B  29     -33.458   2.854  -5.553  1.00 59.40           N  \\nATOM    407  N   ALA B  30     -28.950   7.446  -5.884  1.00 72.05           N  \\nATOM    408  CA  ALA B  30     -28.366   8.568  -6.616  1.00 74.07           C  \\nATOM    409  C   ALA B  30     -27.782   8.115  -7.955  1.00 73.01           C  \\nATOM    410  O   ALA B  30     -27.485   8.928  -8.831  1.00 64.47           O  \\nATOM    411  CB  ALA B  30     -27.302   9.269  -5.772  1.00 56.14           C  \\nATOM    412  OXT ALA B  30     -27.594   6.921  -8.197  1.00 73.19           O  \\nTER     413      ALA B  30                                                      \\nHETATM  414  O   HOH A  22     -29.500  -7.696 -14.149  1.00 50.68           O  \\nHETATM  415  O   HOH A  23     -11.374  10.214 -16.391  1.00 40.62           O  \\nHETATM  416  O   HOH A  24     -11.187   9.560 -19.007  1.00 44.09           O  \\nHETATM  417  O   HOH A  25     -22.708  -0.422 -18.245  1.00 25.03           O  \\nHETATM  418  O   HOH A  26     -17.282   8.923   0.122  1.00 44.14           O  \\nHETATM  419  O   HOH A  27     -27.802  -5.289 -20.151  1.00 41.02           O  \\nHETATM  420  O   HOH A  28     -13.732   9.674 -11.684  1.00 48.35           O  \\nHETATM  421  O   HOH A  29     -26.166   6.685 -14.912  1.00 50.84           O  \\nHETATM  422  O   HOH A  30     -30.826  -5.812 -17.868  1.00 54.11           O  \\nHETATM  423  O   HOH A  31     -14.803   9.989  -8.715  1.00 62.79           O  \\nHETATM  424  O   HOH A  32     -19.609   9.964  -9.626  1.00 46.30           O  \\nHETATM  425  O   HOH A  33     -13.690  11.749 -14.300  1.00 49.90           O  \\nHETATM  426  O   HOH B  31     -24.729  10.263 -12.014  1.00 61.01           O  \\nHETATM  427  O   HOH B  32     -20.724 -10.273 -19.319  1.00 52.72           O  \\nHETATM  428  O   HOH B  33     -17.700 -13.422  -2.674  1.00 25.54           O  \\nHETATM  429  O   HOH B  34     -28.494  10.301 -14.536  1.00 56.88           O  \\nHETATM  430  O   HOH B  35     -24.545   6.027  -8.162  1.00 32.84           O  \\nHETATM  431  O   HOH B  36     -27.182  -2.924  -7.024  1.00 27.65           O  \\nHETATM  432  O   HOH B  37     -24.782  -9.599  -9.935  1.00 26.60           O  \\nHETATM  433  O   HOH B  38     -14.505  -9.421 -15.869  1.00 44.83           O  \\nHETATM  434  O   HOH B  39     -28.442  -1.829  -2.831  1.00 43.18           O  \\nHETATM  435  O   HOH B  40     -12.046  -0.492  -0.930  1.00 25.29           O  \\nHETATM  436  O   HOH B  41     -18.702  -9.557  -7.601  1.00 29.84           O  \\nHETATM  437  O   HOH B  42      -8.200  -0.632  -9.134  1.00 32.65           O  \\nHETATM  438  O   HOH B  43     -15.277 -10.853  -7.465  1.00 43.14           O  \\nHETATM  439  O   HOH B  44     -22.107   4.402  -6.065  1.00 31.09           O  \\nHETATM  440  O   HOH B  45      -5.206   0.010  -2.235  1.00 43.01           O  \\nHETATM  441  O   HOH B  46     -26.729  -9.715  -6.146  1.00 30.80           O  \\nHETATM  442  O   HOH B  47     -27.875   3.972  -9.042  1.00 40.15           O  \\nHETATM  443  O   HOH B  48     -20.453  -7.642  -0.550  1.00 34.77           O  \\nHETATM  444  O   HOH B  49     -17.760 -10.041  -0.552  1.00 37.12           O  \\nHETATM  445  O   HOH B  50     -22.932 -10.505 -16.556  1.00 47.78           O  \\nHETATM  446  O   HOH B  51     -28.195   5.830  -2.571  1.00 53.39           O  \\nHETATM  447  O   HOH B  52     -17.118 -11.123 -18.586  1.00 52.94           O  \\nHETATM  448  O   HOH B  53     -27.581   8.994 -11.659  1.00 53.89           O  \\nCONECT   43   76                                                                \\nCONECT   49  231                                                                \\nCONECT   76   43                                                                \\nCONECT  162  321                                                                \\nCONECT  231   49                                                                \\nCONECT  321  162                                                                \\nMASTER      278    0    0    4    0    0    0    6  438    2    6    5          \\nEND                                                                             \\n\",\"pdb\");\n\tviewer_17563213462829392.setStyle({\"cartoon\": {\"color\": \"spectrum\"}});\n\tviewer_17563213462829392.zoomTo();\nviewer_17563213462829392.render();\n});\n</script>",
      "text/html": [
       "<div id=\"3dmolviewer_17563213462829392\"  style=\"position: relative; width: 400px; height: 300px;\">\n",
       "        <p id=\"3dmolwarning_17563213462829392\" style=\"background-color:#ffcccc;color:black\">3Dmol.js failed to load for some reason.  Please check your browser console for error messages.<br></p>\n",
       "        </div>\n",
       "<script>\n",
       "\n",
       "var loadScriptAsync = function(uri){\n",
       "  return new Promise((resolve, reject) => {\n",
       "    //this is to ignore the existence of requirejs amd\n",
       "    var savedexports, savedmodule;\n",
       "    if (typeof exports !== 'undefined') savedexports = exports;\n",
       "    else exports = {}\n",
       "    if (typeof module !== 'undefined') savedmodule = module;\n",
       "    else module = {}\n",
       "\n",
       "    var tag = document.createElement('script');\n",
       "    tag.src = uri;\n",
       "    tag.async = true;\n",
       "    tag.onload = () => {\n",
       "        exports = savedexports;\n",
       "        module = savedmodule;\n",
       "        resolve();\n",
       "    };\n",
       "  var firstScriptTag = document.getElementsByTagName('script')[0];\n",
       "  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n",
       "});\n",
       "};\n",
       "\n",
       "if(typeof $3Dmolpromise === 'undefined') {\n",
       "$3Dmolpromise = null;\n",
       "  $3Dmolpromise = loadScriptAsync('https://cdn.jsdelivr.net/npm/3dmol@2.5.2/build/3Dmol-min.js');\n",
       "}\n",
       "\n",
       "var viewer_17563213462829392 = null;\n",
       "var warn = document.getElementById(\"3dmolwarning_17563213462829392\");\n",
       "if(warn) {\n",
       "    warn.parentNode.removeChild(warn);\n",
       "}\n",
       "$3Dmolpromise.then(function() {\n",
       "viewer_17563213462829392 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_17563213462829392\"),{backgroundColor:\"white\"});\n",
       "viewer_17563213462829392.zoomTo();\n",
       "\tviewer_17563213462829392.addModel(\"HEADER    HORMONE                                 01-JUL-09   3I40              \\nTITLE     HUMAN INSULIN                                                         \\nCOMPND    MOL_ID: 1;                                                            \\nCOMPND   2 MOLECULE: INSULIN A CHAIN;                                           \\nCOMPND   3 CHAIN: A;                                                            \\nCOMPND   4 ENGINEERED: YES;                                                     \\nCOMPND   5 MOL_ID: 2;                                                           \\nCOMPND   6 MOLECULE: INSULIN B CHAIN;                                           \\nCOMPND   7 CHAIN: B;                                                            \\nCOMPND   8 ENGINEERED: YES                                                      \\nSOURCE    MOL_ID: 1;                                                            \\nSOURCE   2 ORGANISM_SCIENTIFIC: HOMO SAPIENS;                                   \\nSOURCE   3 ORGANISM_COMMON: HUMAN;                                              \\nSOURCE   4 ORGANISM_TAXID: 9606;                                                \\nSOURCE   5 EXPRESSION_SYSTEM: ESCHERICHIA COLI;                                 \\nSOURCE   6 EXPRESSION_SYSTEM_TAXID: 562;                                        \\nSOURCE   7 MOL_ID: 2;                                                           \\nSOURCE   8 ORGANISM_SCIENTIFIC: HOMO SAPIENS;                                   \\nSOURCE   9 ORGANISM_COMMON: HUMAN;                                              \\nSOURCE  10 ORGANISM_TAXID: 9606;                                                \\nSOURCE  11 EXPRESSION_SYSTEM: ESCHERICHIA COLI;                                 \\nSOURCE  12 EXPRESSION_SYSTEM_TAXID: 562                                         \\nKEYWDS    INSULIN, CARBOHYDRATE METABOLISM, CLEAVAGE ON PAIR OF BASIC RESIDUES, \\nKEYWDS   2 DIABETES MELLITUS, DISEASE MUTATION, DISULFIDE BOND, GLUCOSE         \\nKEYWDS   3 METABOLISM, HORMONE, SECRETED                                        \\nEXPDTA    X-RAY DIFFRACTION                                                     \\nAUTHOR    V.I.TIMOFEEV,V.V.BEZUGLOV,K.A.MIROSHNIKOV,R.N.CHUPROV-NETOCHIN,       \\nAUTHOR   2 I.P.KURANOVA                                                         \\nREVDAT   4   06-NOV-24 3I40    1       REMARK                                   \\nREVDAT   3   01-NOV-23 3I40    1       REMARK                                   \\nREVDAT   2   05-MAY-10 3I40    1       JRNL                                     \\nREVDAT   1   05-JAN-10 3I40    0                                                \\nJRNL        AUTH   V.I.TIMOFEEV,R.N.CHUPROV-NETOCHIN,V.R.SAMIGINA,V.V.BEZUGLOV, \\nJRNL        AUTH 2 K.A.MIROSHNIKOV,I.P.KURANOVA                                 \\nJRNL        TITL   X-RAY INVESTIGATION OF GENE-ENGINEERED HUMAN INSULIN         \\nJRNL        TITL 2 CRYSTALLIZED FROM A SOLUTION CONTAINING POLYSIALIC ACID.     \\nJRNL        REF    ACTA CRYSTALLOGR.,SECT.F      V.  66   259 2010              \\nJRNL        REFN                   ESSN 1744-3091                               \\nJRNL        PMID   20208155                                                     \\nJRNL        DOI    10.1107/S1744309110000461                                    \\nREMARK   2                                                                      \\nREMARK   2 RESOLUTION.    1.85 ANGSTROMS.                                       \\nREMARK   3                                                                      \\nREMARK   3 REFINEMENT.                                                          \\nREMARK   3   PROGRAM     : PHENIX (PHENIX.REFINE)                               \\nREMARK   3   AUTHORS     : PAUL ADAMS,PAVEL AFONINE,VINCENT CHEN,IAN            \\nREMARK   3               : DAVIS,KRESHNA GOPAL,RALF GROSSE-KUNSTLEVE,           \\nREMARK   3               : LI-WEI HUNG,ROBERT IMMORMINO,TOM IOERGER,            \\nREMARK   3               : AIRLIE MCCOY,ERIK MCKEE,NIGEL MORIARTY,              \\nREMARK   3               : REETAL PAI,RANDY READ,JANE RICHARDSON,               \\nREMARK   3               : DAVID RICHARDSON,TOD ROMO,JIM SACCHETTINI,           \\nREMARK   3               : NICHOLAS SAUTER,JACOB SMITH,LAURENT                  \\nREMARK   3               : STORONI,TOM TERWILLIGER,PETER ZWART                  \\nREMARK   3                                                                      \\nREMARK   3    REFINEMENT TARGET : ML                                            \\nREMARK   3                                                                      \\nREMARK   3  DATA USED IN REFINEMENT.                                            \\nREMARK   3   RESOLUTION RANGE HIGH (ANGSTROMS) : 1.85                           \\nREMARK   3   RESOLUTION RANGE LOW  (ANGSTROMS) : 31.53                          \\nREMARK   3   MIN(FOBS/SIGMA_FOBS)              : 0.000                          \\nREMARK   3   COMPLETENESS FOR RANGE        (%) : 99.1                           \\nREMARK   3   NUMBER OF REFLECTIONS             : 6640                           \\nREMARK   3                                                                      \\nREMARK   3  FIT TO DATA USED IN REFINEMENT.                                     \\nREMARK   3   R VALUE     (WORKING + TEST SET) : 0.202                           \\nREMARK   3   R VALUE            (WORKING SET) : 0.200                           \\nREMARK   3   FREE R VALUE                     : 0.228                           \\nREMARK   3   FREE R VALUE TEST SET SIZE   (%) : 4.760                           \\nREMARK   3   FREE R VALUE TEST SET COUNT      : 316                             \\nREMARK   3                                                                      \\nREMARK   3  FIT TO DATA USED IN REFINEMENT (IN BINS).                           \\nREMARK   3   BIN  RESOLUTION RANGE  COMPL.    NWORK NFREE   RWORK  RFREE        \\nREMARK   3     1 31.5295 -  2.3309    1.00     3219   154  0.1910 0.2145        \\nREMARK   3     2  2.3309 -  1.8501    0.99     3105   162  0.2301 0.2627        \\nREMARK   3                                                                      \\nREMARK   3  BULK SOLVENT MODELLING.                                             \\nREMARK   3   METHOD USED        : FLAT BULK SOLVENT MODEL                       \\nREMARK   3   SOLVENT RADIUS     : 1.11                                          \\nREMARK   3   SHRINKAGE RADIUS   : 0.90                                          \\nREMARK   3   K_SOL              : 0.35                                          \\nREMARK   3   B_SOL              : 47.05                                         \\nREMARK   3                                                                      \\nREMARK   3  ERROR ESTIMATES.                                                    \\nREMARK   3   COORDINATE ERROR (MAXIMUM-LIKELIHOOD BASED)     : 0.240            \\nREMARK   3   PHASE ERROR (DEGREES, MAXIMUM-LIKELIHOOD BASED) : 24.930           \\nREMARK   3                                                                      \\nREMARK   3  B VALUES.                                                           \\nREMARK   3   FROM WILSON PLOT           (A**2) : 24.04                          \\nREMARK   3   MEAN B VALUE      (OVERALL, A**2) : 30.30                          \\nREMARK   3   OVERALL ANISOTROPIC B VALUE.                                       \\nREMARK   3    B11 (A**2) : NULL                                                 \\nREMARK   3    B22 (A**2) : NULL                                                 \\nREMARK   3    B33 (A**2) : NULL                                                 \\nREMARK   3    B12 (A**2) : NULL                                                 \\nREMARK   3    B13 (A**2) : NULL                                                 \\nREMARK   3    B23 (A**2) : NULL                                                 \\nREMARK   3                                                                      \\nREMARK   3  TWINNING INFORMATION.                                               \\nREMARK   3   FRACTION: NULL                                                     \\nREMARK   3   OPERATOR: NULL                                                     \\nREMARK   3                                                                      \\nREMARK   3  DEVIATIONS FROM IDEAL VALUES.                                       \\nREMARK   3                 RMSD          COUNT                                  \\nREMARK   3   BOND      :  0.005            423                                  \\nREMARK   3   ANGLE     :  0.760            569                                  \\nREMARK   3   CHIRALITY :  0.068             62                                  \\nREMARK   3   PLANARITY :  0.002             73                                  \\nREMARK   3   DIHEDRAL  : 17.205            141                                  \\nREMARK   3                                                                      \\nREMARK   3  TLS DETAILS                                                         \\nREMARK   3   NUMBER OF TLS GROUPS  : NULL                                       \\nREMARK   3                                                                      \\nREMARK   3  NCS DETAILS                                                         \\nREMARK   3   NUMBER OF NCS GROUPS : NULL                                        \\nREMARK   3                                                                      \\nREMARK   3  OTHER REFINEMENT REMARKS: NULL                                      \\nREMARK   4                                                                      \\nREMARK   4 3I40 COMPLIES WITH FORMAT V. 3.30, 13-JUL-11                         \\nREMARK 100                                                                      \\nREMARK 100 THIS ENTRY HAS BEEN PROCESSED BY PDBJ ON 06-JUL-09.                  \\nREMARK 100 THE DEPOSITION ID IS D_1000053935.                                   \\nREMARK 200                                                                      \\nREMARK 200 EXPERIMENTAL DETAILS                                                 \\nREMARK 200  EXPERIMENT TYPE                : X-RAY DIFFRACTION                  \\nREMARK 200  DATE OF DATA COLLECTION        : 10-MAR-09                          \\nREMARK 200  TEMPERATURE           (KELVIN) : 100                                \\nREMARK 200  PH                             : NULL                               \\nREMARK 200  NUMBER OF CRYSTALS USED        : 1                                  \\nREMARK 200                                                                      \\nREMARK 200  SYNCHROTRON              (Y/N) : N                                  \\nREMARK 200  RADIATION SOURCE               : ROTATING ANODE                     \\nREMARK 200  BEAMLINE                       : NULL                               \\nREMARK 200  X-RAY GENERATOR MODEL          : BRUKER AXS MICROSTAR               \\nREMARK 200  MONOCHROMATIC OR LAUE    (M/L) : M                                  \\nREMARK 200  WAVELENGTH OR RANGE        (A) : NULL                               \\nREMARK 200  MONOCHROMATOR                  : NULL                               \\nREMARK 200  OPTICS                         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  DETECTOR TYPE                  : CCD                                \\nREMARK 200  DETECTOR MANUFACTURER          : MAR CCD 165 MM                     \\nREMARK 200  INTENSITY-INTEGRATION SOFTWARE : DENZO                              \\nREMARK 200  DATA SCALING SOFTWARE          : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  NUMBER OF UNIQUE REFLECTIONS   : 6677                               \\nREMARK 200  RESOLUTION RANGE HIGH      (A) : 1.850                              \\nREMARK 200  RESOLUTION RANGE LOW       (A) : 54.636                             \\nREMARK 200  REJECTION CRITERIA  (SIGMA(I)) : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 OVERALL.                                                             \\nREMARK 200  COMPLETENESS FOR RANGE     (%) : NULL                               \\nREMARK 200  DATA REDUNDANCY                : NULL                               \\nREMARK 200  R MERGE                    (I) : NULL                               \\nREMARK 200  R SYM                      (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR THE DATA SET  : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 IN THE HIGHEST RESOLUTION SHELL.                                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE HIGH (A) : NULL                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE LOW  (A) : NULL                     \\nREMARK 200  COMPLETENESS FOR SHELL     (%) : NULL                               \\nREMARK 200  DATA REDUNDANCY IN SHELL       : NULL                               \\nREMARK 200  R MERGE FOR SHELL          (I) : NULL                               \\nREMARK 200  R SYM FOR SHELL            (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR SHELL         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 DIFFRACTION PROTOCOL: SINGLE WAVELENGTH                              \\nREMARK 200 METHOD USED TO DETERMINE THE STRUCTURE: MOLECULAR REPLACEMENT        \\nREMARK 200 SOFTWARE USED: NULL                                                  \\nREMARK 200 STARTING MODEL: PDB ENTRY 2G4M                                       \\nREMARK 200                                                                      \\nREMARK 200 REMARK: NULL                                                         \\nREMARK 280                                                                      \\nREMARK 280 CRYSTAL                                                              \\nREMARK 280 SOLVENT CONTENT, VS   (%): 62.90                                     \\nREMARK 280 MATTHEWS COEFFICIENT, VM (ANGSTROMS**3/DA): 3.31                     \\nREMARK 280                                                                      \\nREMARK 280 CRYSTALLIZATION CONDITIONS: NULL                                     \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY                                            \\nREMARK 290 SYMMETRY OPERATORS FOR SPACE GROUP: I 21 3                           \\nREMARK 290                                                                      \\nREMARK 290      SYMOP   SYMMETRY                                                \\nREMARK 290     NNNMMM   OPERATOR                                                \\nREMARK 290       1555   X,Y,Z                                                   \\nREMARK 290       2555   -X+1/2,-Y,Z+1/2                                         \\nREMARK 290       3555   -X,Y+1/2,-Z+1/2                                         \\nREMARK 290       4555   X+1/2,-Y+1/2,-Z                                         \\nREMARK 290       5555   Z,X,Y                                                   \\nREMARK 290       6555   Z+1/2,-X+1/2,-Y                                         \\nREMARK 290       7555   -Z+1/2,-X,Y+1/2                                         \\nREMARK 290       8555   -Z,X+1/2,-Y+1/2                                         \\nREMARK 290       9555   Y,Z,X                                                   \\nREMARK 290      10555   -Y,Z+1/2,-X+1/2                                         \\nREMARK 290      11555   Y+1/2,-Z+1/2,-X                                         \\nREMARK 290      12555   -Y+1/2,-Z,X+1/2                                         \\nREMARK 290      13555   X+1/2,Y+1/2,Z+1/2                                       \\nREMARK 290      14555   -X,-Y+1/2,Z                                             \\nREMARK 290      15555   -X+1/2,Y,-Z                                             \\nREMARK 290      16555   X,-Y,-Z+1/2                                             \\nREMARK 290      17555   Z+1/2,X+1/2,Y+1/2                                       \\nREMARK 290      18555   Z,-X,-Y+1/2                                             \\nREMARK 290      19555   -Z,-X+1/2,Y                                             \\nREMARK 290      20555   -Z+1/2,X,-Y                                             \\nREMARK 290      21555   Y+1/2,Z+1/2,X+1/2                                       \\nREMARK 290      22555   -Y+1/2,Z,-X                                             \\nREMARK 290      23555   Y,-Z,-X+1/2                                             \\nREMARK 290      24555   -Y,-Z+1/2,X                                             \\nREMARK 290                                                                      \\nREMARK 290     WHERE NNN -> OPERATOR NUMBER                                     \\nREMARK 290           MMM -> TRANSLATION VECTOR                                  \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY TRANSFORMATIONS                            \\nREMARK 290 THE FOLLOWING TRANSFORMATIONS OPERATE ON THE ATOM/HETATM             \\nREMARK 290 RECORDS IN THIS ENTRY TO PRODUCE CRYSTALLOGRAPHICALLY                \\nREMARK 290 RELATED MOLECULES.                                                   \\nREMARK 290   SMTRY1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY1   2 -1.000000  0.000000  0.000000       38.61000            \\nREMARK 290   SMTRY2   2  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   2  0.000000  0.000000  1.000000       38.61000            \\nREMARK 290   SMTRY1   3 -1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   3  0.000000  1.000000  0.000000       38.61000            \\nREMARK 290   SMTRY3   3  0.000000  0.000000 -1.000000       38.61000            \\nREMARK 290   SMTRY1   4  1.000000  0.000000  0.000000       38.61000            \\nREMARK 290   SMTRY2   4  0.000000 -1.000000  0.000000       38.61000            \\nREMARK 290   SMTRY3   4  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290   SMTRY1   5  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY2   5  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   5  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY1   6  0.000000  0.000000  1.000000       38.61000            \\nREMARK 290   SMTRY2   6 -1.000000  0.000000  0.000000       38.61000            \\nREMARK 290   SMTRY3   6  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY1   7  0.000000  0.000000 -1.000000       38.61000            \\nREMARK 290   SMTRY2   7 -1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   7  0.000000  1.000000  0.000000       38.61000            \\nREMARK 290   SMTRY1   8  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290   SMTRY2   8  1.000000  0.000000  0.000000       38.61000            \\nREMARK 290   SMTRY3   8  0.000000 -1.000000  0.000000       38.61000            \\nREMARK 290   SMTRY1   9  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   9  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY3   9  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY1  10  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2  10  0.000000  0.000000  1.000000       38.61000            \\nREMARK 290   SMTRY3  10 -1.000000  0.000000  0.000000       38.61000            \\nREMARK 290   SMTRY1  11  0.000000  1.000000  0.000000       38.61000            \\nREMARK 290   SMTRY2  11  0.000000  0.000000 -1.000000       38.61000            \\nREMARK 290   SMTRY3  11 -1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY1  12  0.000000 -1.000000  0.000000       38.61000            \\nREMARK 290   SMTRY2  12  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290   SMTRY3  12  1.000000  0.000000  0.000000       38.61000            \\nREMARK 290   SMTRY1  13  1.000000  0.000000  0.000000       38.61000            \\nREMARK 290   SMTRY2  13  0.000000  1.000000  0.000000       38.61000            \\nREMARK 290   SMTRY3  13  0.000000  0.000000  1.000000       38.61000            \\nREMARK 290   SMTRY1  14 -1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2  14  0.000000 -1.000000  0.000000       38.61000            \\nREMARK 290   SMTRY3  14  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY1  15 -1.000000  0.000000  0.000000       38.61000            \\nREMARK 290   SMTRY2  15  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3  15  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290   SMTRY1  16  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2  16  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3  16  0.000000  0.000000 -1.000000       38.61000            \\nREMARK 290   SMTRY1  17  0.000000  0.000000  1.000000       38.61000            \\nREMARK 290   SMTRY2  17  1.000000  0.000000  0.000000       38.61000            \\nREMARK 290   SMTRY3  17  0.000000  1.000000  0.000000       38.61000            \\nREMARK 290   SMTRY1  18  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY2  18 -1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3  18  0.000000 -1.000000  0.000000       38.61000            \\nREMARK 290   SMTRY1  19  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290   SMTRY2  19 -1.000000  0.000000  0.000000       38.61000            \\nREMARK 290   SMTRY3  19  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY1  20  0.000000  0.000000 -1.000000       38.61000            \\nREMARK 290   SMTRY2  20  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3  20  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY1  21  0.000000  1.000000  0.000000       38.61000            \\nREMARK 290   SMTRY2  21  0.000000  0.000000  1.000000       38.61000            \\nREMARK 290   SMTRY3  21  1.000000  0.000000  0.000000       38.61000            \\nREMARK 290   SMTRY1  22  0.000000 -1.000000  0.000000       38.61000            \\nREMARK 290   SMTRY2  22  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY3  22 -1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY1  23  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2  23  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290   SMTRY3  23 -1.000000  0.000000  0.000000       38.61000            \\nREMARK 290   SMTRY1  24  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2  24  0.000000  0.000000 -1.000000       38.61000            \\nREMARK 290   SMTRY3  24  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290                                                                      \\nREMARK 290 REMARK: NULL                                                         \\nREMARK 300                                                                      \\nREMARK 300 BIOMOLECULE: 1                                                       \\nREMARK 300 SEE REMARK 350 FOR THE AUTHOR PROVIDED AND/OR PROGRAM                \\nREMARK 300 GENERATED ASSEMBLY INFORMATION FOR THE STRUCTURE IN                  \\nREMARK 300 THIS ENTRY. THE REMARK MAY ALSO PROVIDE INFORMATION ON               \\nREMARK 300 BURIED SURFACE AREA.                                                 \\nREMARK 350                                                                      \\nREMARK 350 COORDINATES FOR A COMPLETE MULTIMER REPRESENTING THE KNOWN           \\nREMARK 350 BIOLOGICALLY SIGNIFICANT OLIGOMERIZATION STATE OF THE                \\nREMARK 350 MOLECULE CAN BE GENERATED BY APPLYING BIOMT TRANSFORMATIONS          \\nREMARK 350 GIVEN BELOW.  BOTH NON-CRYSTALLOGRAPHIC AND                          \\nREMARK 350 CRYSTALLOGRAPHIC OPERATIONS ARE GIVEN.                               \\nREMARK 350                                                                      \\nREMARK 350 BIOMOLECULE: 1                                                       \\nREMARK 350 AUTHOR DETERMINED BIOLOGICAL UNIT: DIMERIC                           \\nREMARK 350 SOFTWARE DETERMINED QUATERNARY STRUCTURE: DIMERIC                    \\nREMARK 350 SOFTWARE USED: PISA                                                  \\nREMARK 350 TOTAL BURIED SURFACE AREA: 1520 ANGSTROM**2                          \\nREMARK 350 SURFACE AREA OF THE COMPLEX: 3380 ANGSTROM**2                        \\nREMARK 350 CHANGE IN SOLVENT FREE ENERGY: -14.0 KCAL/MOL                        \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: A, B                                  \\nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 900                                                                      \\nREMARK 900 RELATED ENTRIES                                                      \\nREMARK 900 RELATED ID: 3I3Z   RELATED DB: PDB                                   \\nREMARK 999                                                                      \\nREMARK 999 SEQUENCE                                                             \\nREMARK 999 THE CONFLICT OF THE SEQUENCE MAY DUE TO ARTIFACT.                    \\nDBREF  3I40 A    1    21  UNP    P01308   INS_HUMAN       90    110             \\nDBREF  3I40 B    1    30  UNP    P01308   INS_HUMAN       25     54             \\nSEQADV 3I40 ALA B   30  UNP  P01308    THR    54 SEE REMARK 999                 \\nSEQRES   1 A   21  GLY ILE VAL GLU GLN CYS CYS THR SER ILE CYS SER LEU          \\nSEQRES   2 A   21  TYR GLN LEU GLU ASN TYR CYS ASN                              \\nSEQRES   1 B   30  PHE VAL ASN GLN HIS LEU CYS GLY SER HIS LEU VAL GLU          \\nSEQRES   2 B   30  ALA LEU TYR LEU VAL CYS GLY GLU ARG GLY PHE PHE TYR          \\nSEQRES   3 B   30  THR PRO LYS ALA                                              \\nFORMUL   3  HOH   *35(H2 O)                                                     \\nHELIX    1   1 GLY A    1  CYS A    7  1                                   7    \\nHELIX    2   2 SER A   12  ASN A   18  1                                   7    \\nHELIX    3   3 GLY B    8  GLY B   20  1                                  13    \\nHELIX    4   4 GLU B   21  GLY B   23  5                                   3    \\nSSBOND   1 CYS A    6    CYS A   11                          1555   1555  2.03  \\nSSBOND   2 CYS A    7    CYS B    7                          1555   1555  2.04  \\nSSBOND   3 CYS A   20    CYS B   19                          1555   1555  2.04  \\nCRYST1   77.220   77.220   77.220  90.00  90.00  90.00 I 21 3       24          \\nORIGX1      1.000000  0.000000  0.000000        0.00000                         \\nORIGX2      0.000000  1.000000  0.000000        0.00000                         \\nORIGX3      0.000000  0.000000  1.000000        0.00000                         \\nSCALE1      0.012950  0.000000  0.000000        0.00000                         \\nSCALE2      0.000000  0.012950  0.000000        0.00000                         \\nSCALE3      0.000000  0.000000  0.012950        0.00000                         \\nATOM      1  N   GLY A   1     -27.279   6.238 -12.314  1.00 45.01           N  \\nATOM      2  CA  GLY A   1     -26.249   6.028 -11.313  1.00 43.47           C  \\nATOM      3  C   GLY A   1     -25.582   4.677 -11.471  1.00 34.37           C  \\nATOM      4  O   GLY A   1     -25.731   4.023 -12.501  1.00 30.09           O  \\nATOM      5  N   ILE A   2     -24.853   4.248 -10.446  1.00 32.71           N  \\nATOM      6  CA  ILE A   2     -24.070   3.023 -10.550  1.00 22.60           C  \\nATOM      7  C   ILE A   2     -24.915   1.786 -10.840  1.00 23.32           C  \\nATOM      8  O   ILE A   2     -24.469   0.865 -11.529  1.00 26.36           O  \\nATOM      9  CB  ILE A   2     -23.208   2.791  -9.302  1.00 28.24           C  \\nATOM     10  CG1 ILE A   2     -22.249   1.620  -9.538  1.00 25.03           C  \\nATOM     11  CG2 ILE A   2     -24.090   2.579  -8.068  1.00 32.70           C  \\nATOM     12  CD1 ILE A   2     -21.209   1.479  -8.457  1.00 30.60           C  \\nATOM     13  N   VAL A   3     -26.137   1.755 -10.324  1.00 27.13           N  \\nATOM     14  CA  VAL A   3     -26.993   0.599 -10.563  1.00 30.10           C  \\nATOM     15  C   VAL A   3     -27.324   0.471 -12.049  1.00 32.71           C  \\nATOM     16  O   VAL A   3     -27.258  -0.617 -12.621  1.00 28.83           O  \\nATOM     17  CB  VAL A   3     -28.284   0.657  -9.740  1.00 30.51           C  \\nATOM     18  CG1 VAL A   3     -29.194  -0.497 -10.127  1.00 35.07           C  \\nATOM     19  CG2 VAL A   3     -27.956   0.613  -8.242  1.00 27.53           C  \\nATOM     20  N   GLU A   4     -27.669   1.596 -12.667  1.00 34.97           N  \\nATOM     21  CA  GLU A   4     -28.001   1.627 -14.086  1.00 31.24           C  \\nATOM     22  C   GLU A   4     -26.797   1.224 -14.927  1.00 32.52           C  \\nATOM     23  O   GLU A   4     -26.933   0.517 -15.924  1.00 31.47           O  \\nATOM     24  CB  GLU A   4     -28.474   3.027 -14.485  1.00 38.19           C  \\nATOM     25  CG  GLU A   4     -29.829   3.420 -13.908  1.00 61.51           C  \\nATOM     26  CD  GLU A   4     -29.832   3.535 -12.387  1.00 66.16           C  \\nATOM     27  OE1 GLU A   4     -28.868   4.094 -11.814  1.00 42.60           O  \\nATOM     28  OE2 GLU A   4     -30.812   3.071 -11.762  1.00 67.86           O  \\nATOM     29  N   GLN A   5     -25.615   1.664 -14.508  1.00 28.29           N  \\nATOM     30  CA  GLN A   5     -24.393   1.410 -15.266  1.00 31.01           C  \\nATOM     31  C   GLN A   5     -23.776   0.028 -15.056  1.00 39.69           C  \\nATOM     32  O   GLN A   5     -23.149  -0.511 -15.967  1.00 26.39           O  \\nATOM     33  CB  GLN A   5     -23.349   2.492 -14.974  1.00 25.23           C  \\nATOM     34  CG  GLN A   5     -23.818   3.887 -15.346  1.00 38.13           C  \\nATOM     35  CD  GLN A   5     -24.535   3.921 -16.693  1.00 53.62           C  \\nATOM     36  OE1 GLN A   5     -24.074   3.334 -17.674  1.00 38.85           O  \\nATOM     37  NE2 GLN A   5     -25.671   4.609 -16.741  1.00 42.57           N  \\nATOM     38  N   CYS A   6     -23.940  -0.551 -13.868  1.00 22.46           N  \\nATOM     39  CA  CYS A   6     -23.214  -1.776 -13.549  1.00 20.37           C  \\nATOM     40  C   CYS A   6     -24.114  -2.963 -13.225  1.00 18.21           C  \\nATOM     41  O   CYS A   6     -23.677  -4.107 -13.293  1.00 21.83           O  \\nATOM     42  CB  CYS A   6     -22.235  -1.524 -12.396  1.00 22.32           C  \\nATOM     43  SG  CYS A   6     -20.756  -0.602 -12.893  1.00 27.82           S  \\nATOM     44  N   CYS A   7     -25.359  -2.685 -12.860  1.00 22.16           N  \\nATOM     45  CA  CYS A   7     -26.307  -3.744 -12.520  1.00 21.25           C  \\nATOM     46  C   CYS A   7     -27.245  -4.032 -13.690  1.00 31.44           C  \\nATOM     47  O   CYS A   7     -27.307  -5.155 -14.190  1.00 24.09           O  \\nATOM     48  CB  CYS A   7     -27.119  -3.351 -11.286  1.00 22.97           C  \\nATOM     49  SG  CYS A   7     -28.355  -4.584 -10.791  1.00 28.27           S  \\nATOM     50  N   THR A   8     -27.967  -3.001 -14.122  1.00 29.15           N  \\nATOM     51  CA  THR A   8     -28.901  -3.121 -15.236  1.00 29.59           C  \\nATOM     52  C   THR A   8     -28.147  -3.334 -16.542  1.00 34.52           C  \\nATOM     53  O   THR A   8     -28.647  -3.986 -17.454  1.00 38.61           O  \\nATOM     54  CB  THR A   8     -29.780  -1.868 -15.371  1.00 31.24           C  \\nATOM     55  OG1 THR A   8     -30.023  -1.303 -14.080  1.00 41.12           O  \\nATOM     56  CG2 THR A   8     -31.100  -2.218 -16.029  1.00 51.07           C  \\nATOM     57  N   SER A   9     -26.948  -2.769 -16.630  1.00 26.15           N  \\nATOM     58  CA  SER A   9     -26.064  -3.019 -17.765  1.00 21.41           C  \\nATOM     59  C   SER A   9     -24.699  -3.529 -17.309  1.00 32.36           C  \\nATOM     60  O   SER A   9     -24.419  -3.594 -16.111  1.00 29.40           O  \\nATOM     61  CB  SER A   9     -25.910  -1.765 -18.626  1.00 32.53           C  \\nATOM     62  OG  SER A   9     -25.465  -0.670 -17.851  1.00 47.07           O  \\nATOM     63  N   ILE A  10     -23.849  -3.891 -18.261  1.00 24.43           N  \\nATOM     64  CA  ILE A  10     -22.562  -4.492 -17.924  1.00 24.22           C  \\nATOM     65  C   ILE A  10     -21.591  -3.454 -17.367  1.00 30.49           C  \\nATOM     66  O   ILE A  10     -21.420  -2.380 -17.934  1.00 26.28           O  \\nATOM     67  CB  ILE A  10     -21.942  -5.216 -19.128  1.00 26.99           C  \\nATOM     68  CG1 ILE A  10     -22.880  -6.328 -19.595  1.00 26.49           C  \\nATOM     69  CG2 ILE A  10     -20.577  -5.790 -18.770  1.00 25.10           C  \\nATOM     70  CD1 ILE A  10     -22.302  -7.179 -20.689  1.00 44.31           C  \\nATOM     71  N   CYS A  11     -20.972  -3.790 -16.240  1.00 28.14           N  \\nATOM     72  CA  CYS A  11     -20.088  -2.876 -15.534  1.00 25.92           C  \\nATOM     73  C   CYS A  11     -18.752  -2.779 -16.258  1.00 36.06           C  \\nATOM     74  O   CYS A  11     -18.491  -3.524 -17.204  1.00 27.25           O  \\nATOM     75  CB  CYS A  11     -19.867  -3.387 -14.106  1.00 23.76           C  \\nATOM     76  SG  CYS A  11     -19.406  -2.114 -12.908  1.00 28.09           S  \\nATOM     77  N   SER A  12     -17.902  -1.865 -15.807  1.00 25.78           N  \\nATOM     78  CA  SER A  12     -16.554  -1.767 -16.349  1.00 22.91           C  \\nATOM     79  C   SER A  12     -15.634  -1.220 -15.281  1.00 19.20           C  \\nATOM     80  O   SER A  12     -16.076  -0.508 -14.375  1.00 18.62           O  \\nATOM     81  CB  SER A  12     -16.530  -0.845 -17.559  1.00 24.50           C  \\nATOM     82  OG  SER A  12     -16.681   0.502 -17.157  1.00 24.75           O  \\nATOM     83  N   LEU A  13     -14.346  -1.528 -15.392  1.00 22.17           N  \\nATOM     84  CA  LEU A  13     -13.378  -1.013 -14.426  1.00 19.72           C  \\nATOM     85  C   LEU A  13     -13.412   0.508 -14.438  1.00 22.22           C  \\nATOM     86  O   LEU A  13     -13.329   1.158 -13.393  1.00 17.29           O  \\nATOM     87  CB  LEU A  13     -11.971  -1.517 -14.751  1.00 19.22           C  \\nATOM     88  CG  LEU A  13     -10.860  -0.998 -13.840  1.00 17.99           C  \\nATOM     89  CD1 LEU A  13     -11.104  -1.386 -12.388  1.00 18.06           C  \\nATOM     90  CD2 LEU A  13      -9.522  -1.530 -14.308  1.00 15.25           C  \\nATOM     91  N   TYR A  14     -13.550   1.075 -15.632  1.00 19.94           N  \\nATOM     92  CA  TYR A  14     -13.559   2.521 -15.774  1.00 22.72           C  \\nATOM     93  C   TYR A  14     -14.638   3.140 -14.893  1.00 22.64           C  \\nATOM     94  O   TYR A  14     -14.391   4.094 -14.152  1.00 25.33           O  \\nATOM     95  CB ATYR A  14     -13.771   2.911 -17.239  0.50 27.81           C  \\nATOM     96  CB BTYR A  14     -13.786   2.928 -17.233  0.50 27.85           C  \\nATOM     97  CG ATYR A  14     -13.608   4.388 -17.494  0.50 28.31           C  \\nATOM     98  CG BTYR A  14     -13.829   4.427 -17.427  0.50 28.20           C  \\nATOM     99  CD1ATYR A  14     -12.442   4.893 -18.059  0.50 23.25           C  \\nATOM    100  CD1BTYR A  14     -12.714   5.121 -17.879  0.50 25.75           C  \\nATOM    101  CD2ATYR A  14     -14.614   5.282 -17.153  0.50 28.36           C  \\nATOM    102  CD2BTYR A  14     -14.980   5.152 -17.138  0.50 27.68           C  \\nATOM    103  CE1ATYR A  14     -12.289   6.252 -18.283  0.50 27.32           C  \\nATOM    104  CE1BTYR A  14     -12.746   6.496 -18.047  0.50 27.49           C  \\nATOM    105  CE2ATYR A  14     -14.472   6.636 -17.372  0.50 29.32           C  \\nATOM    106  CE2BTYR A  14     -15.021   6.527 -17.300  0.50 28.26           C  \\nATOM    107  CZ ATYR A  14     -13.311   7.117 -17.938  0.50 29.06           C  \\nATOM    108  CZ BTYR A  14     -13.902   7.193 -17.757  0.50 29.57           C  \\nATOM    109  OH ATYR A  14     -13.179   8.469 -18.153  0.50 33.01           O  \\nATOM    110  OH BTYR A  14     -13.938   8.561 -17.922  0.50 32.79           O  \\nATOM    111  N   GLN A  15     -15.837   2.584 -14.969  1.00 21.27           N  \\nATOM    112  CA  GLN A  15     -16.962   3.129 -14.222  1.00 21.51           C  \\nATOM    113  C   GLN A  15     -16.837   2.882 -12.722  1.00 17.30           C  \\nATOM    114  O   GLN A  15     -17.121   3.764 -11.917  1.00 20.82           O  \\nATOM    115  CB  GLN A  15     -18.275   2.561 -14.758  1.00 25.25           C  \\nATOM    116  CG  GLN A  15     -18.527   2.905 -16.214  1.00 32.75           C  \\nATOM    117  CD  GLN A  15     -19.997   3.039 -16.527  1.00 55.05           C  \\nATOM    118  OE1 GLN A  15     -20.680   3.914 -15.987  1.00 53.01           O  \\nATOM    119  NE2 GLN A  15     -20.497   2.177 -17.410  1.00 50.66           N  \\nATOM    120  N   LEU A  16     -16.399   1.687 -12.346  1.00 15.37           N  \\nATOM    121  CA  LEU A  16     -16.236   1.380 -10.930  1.00 17.02           C  \\nATOM    122  C   LEU A  16     -15.301   2.382 -10.273  1.00 16.55           C  \\nATOM    123  O   LEU A  16     -15.565   2.868  -9.172  1.00 19.70           O  \\nATOM    124  CB  LEU A  16     -15.703  -0.037 -10.742  1.00 24.64           C  \\nATOM    125  CG  LEU A  16     -16.709  -1.151 -11.013  1.00 24.23           C  \\nATOM    126  CD1 LEU A  16     -16.007  -2.496 -11.058  1.00 21.90           C  \\nATOM    127  CD2 LEU A  16     -17.786  -1.124  -9.936  1.00 18.48           C  \\nATOM    128  N   GLU A  17     -14.199   2.703 -10.950  1.00 17.86           N  \\nATOM    129  CA  GLU A  17     -13.219   3.608 -10.365  1.00 19.38           C  \\nATOM    130  C   GLU A  17     -13.763   5.026 -10.180  1.00 18.73           C  \\nATOM    131  O   GLU A  17     -13.259   5.782  -9.351  1.00 22.43           O  \\nATOM    132  CB  GLU A  17     -11.910   3.598 -11.155  1.00 20.40           C  \\nATOM    133  CG  GLU A  17     -11.090   2.343 -10.898  1.00 20.17           C  \\nATOM    134  CD  GLU A  17      -9.623   2.514 -11.240  1.00 22.14           C  \\nATOM    135  OE1 GLU A  17      -9.301   3.347 -12.110  1.00 22.42           O  \\nATOM    136  OE2 GLU A  17      -8.792   1.805 -10.640  1.00 27.73           O  \\nATOM    137  N   ASN A  18     -14.799   5.374 -10.932  1.00 20.29           N  \\nATOM    138  CA  ASN A  18     -15.480   6.657 -10.735  1.00 20.00           C  \\nATOM    139  C   ASN A  18     -16.080   6.766  -9.334  1.00 26.36           C  \\nATOM    140  O   ASN A  18     -16.469   7.849  -8.890  1.00 28.46           O  \\nATOM    141  CB  ASN A  18     -16.587   6.839 -11.774  1.00 21.66           C  \\nATOM    142  CG  ASN A  18     -16.061   7.344 -13.113  1.00 25.89           C  \\nATOM    143  OD1 ASN A  18     -15.007   7.974 -13.180  1.00 32.32           O  \\nATOM    144  ND2 ASN A  18     -16.802   7.070 -14.180  1.00 38.57           N  \\nATOM    145  N   TYR A  19     -16.163   5.640  -8.638  1.00 20.84           N  \\nATOM    146  CA  TYR A  19     -16.803   5.618  -7.327  1.00 18.56           C  \\nATOM    147  C   TYR A  19     -15.822   5.394  -6.180  1.00 23.20           C  \\nATOM    148  O   TYR A  19     -16.223   5.337  -5.018  1.00 27.54           O  \\nATOM    149  CB  TYR A  19     -17.945   4.599  -7.298  1.00 20.89           C  \\nATOM    150  CG  TYR A  19     -19.068   4.966  -8.236  1.00 21.62           C  \\nATOM    151  CD1 TYR A  19     -19.115   4.453  -9.521  1.00 23.43           C  \\nATOM    152  CD2 TYR A  19     -20.066   5.848  -7.844  1.00 26.68           C  \\nATOM    153  CE1 TYR A  19     -20.126   4.796 -10.385  1.00 24.39           C  \\nATOM    154  CE2 TYR A  19     -21.082   6.200  -8.708  1.00 31.02           C  \\nATOM    155  CZ  TYR A  19     -21.104   5.672  -9.979  1.00 28.36           C  \\nATOM    156  OH  TYR A  19     -22.114   6.009 -10.853  1.00 42.08           O  \\nATOM    157  N   CYS A  20     -14.534   5.281  -6.499  1.00 21.90           N  \\nATOM    158  CA  CYS A  20     -13.510   5.251  -5.459  1.00 19.66           C  \\nATOM    159  C   CYS A  20     -13.334   6.645  -4.882  1.00 28.69           C  \\nATOM    160  O   CYS A  20     -13.526   7.639  -5.585  1.00 30.53           O  \\nATOM    161  CB  CYS A  20     -12.169   4.783  -6.021  1.00 22.84           C  \\nATOM    162  SG  CYS A  20     -12.209   3.187  -6.829  1.00 22.45           S  \\nATOM    163  N   ASN A  21     -12.955   6.716  -3.609  1.00 26.51           N  \\nATOM    164  CA  ASN A  21     -12.630   7.993  -2.976  1.00 36.60           C  \\nATOM    165  C   ASN A  21     -11.279   8.511  -3.448  1.00 35.58           C  \\nATOM    166  O   ASN A  21     -10.515   7.790  -4.091  1.00 36.84           O  \\nATOM    167  CB  ASN A  21     -12.615   7.863  -1.450  1.00 34.89           C  \\nATOM    168  CG  ASN A  21     -13.978   7.548  -0.878  1.00 33.46           C  \\nATOM    169  OD1 ASN A  21     -14.107   6.726   0.029  1.00 48.60           O  \\nATOM    170  ND2 ASN A  21     -15.008   8.198  -1.408  1.00 39.53           N  \\nATOM    171  OXT ASN A  21     -10.922   9.659  -3.189  1.00 58.00           O  \\nTER     172      ASN A  21                                                      \\nATOM    173  N   PHE B   1     -13.174  -9.740 -13.251  1.00 44.56           N  \\nATOM    174  CA  PHE B   1     -14.387  -9.892 -12.452  1.00 53.84           C  \\nATOM    175  C   PHE B   1     -15.310  -8.684 -12.590  1.00 44.51           C  \\nATOM    176  O   PHE B   1     -16.451  -8.715 -12.133  1.00 40.17           O  \\nATOM    177  CB  PHE B   1     -14.042 -10.126 -10.973  1.00 47.36           C  \\nATOM    178  CG  PHE B   1     -13.222  -9.024 -10.356  1.00 43.05           C  \\nATOM    179  CD1 PHE B   1     -13.816  -8.050  -9.571  1.00 39.13           C  \\nATOM    180  CD2 PHE B   1     -11.854  -8.964 -10.565  1.00 48.11           C  \\nATOM    181  CE1 PHE B   1     -13.059  -7.037  -9.008  1.00 29.34           C  \\nATOM    182  CE2 PHE B   1     -11.093  -7.959 -10.002  1.00 34.97           C  \\nATOM    183  CZ  PHE B   1     -11.696  -6.993  -9.224  1.00 36.24           C  \\nATOM    184  N   VAL B   2     -14.819  -7.623 -13.228  1.00 27.44           N  \\nATOM    185  CA  VAL B   2     -15.568  -6.367 -13.273  1.00 25.65           C  \\nATOM    186  C   VAL B   2     -16.402  -6.197 -14.541  1.00 30.55           C  \\nATOM    187  O   VAL B   2     -17.249  -5.306 -14.628  1.00 30.98           O  \\nATOM    188  CB  VAL B   2     -14.633  -5.163 -13.114  1.00 32.50           C  \\nATOM    189  CG1 VAL B   2     -14.056  -5.139 -11.705  1.00 29.74           C  \\nATOM    190  CG2 VAL B   2     -13.519  -5.227 -14.141  1.00 45.24           C  \\nATOM    191  N   ASN B   3     -16.158  -7.045 -15.528  1.00 28.57           N  \\nATOM    192  CA  ASN B   3     -16.844  -6.904 -16.806  1.00 31.00           C  \\nATOM    193  C   ASN B   3     -18.072  -7.800 -16.928  1.00 29.49           C  \\nATOM    194  O   ASN B   3     -18.115  -8.720 -17.743  1.00 28.57           O  \\nATOM    195  CB  ASN B   3     -15.854  -7.122 -17.946  1.00 31.67           C  \\nATOM    196  CG  ASN B   3     -14.703  -6.139 -17.888  1.00 35.45           C  \\nATOM    197  OD1 ASN B   3     -13.561  -6.507 -17.600  1.00 38.37           O  \\nATOM    198  ND2 ASN B   3     -15.009  -4.871 -18.113  1.00 23.34           N  \\nATOM    199  N   GLN B   4     -19.073  -7.514 -16.099  1.00 31.90           N  \\nATOM    200  CA  GLN B   4     -20.313  -8.287 -16.082  1.00 28.96           C  \\nATOM    201  C   GLN B   4     -21.397  -7.521 -15.328  1.00 23.12           C  \\nATOM    202  O   GLN B   4     -21.154  -6.432 -14.812  1.00 26.67           O  \\nATOM    203  CB  GLN B   4     -20.079  -9.648 -15.425  1.00 33.50           C  \\nATOM    204  CG  GLN B   4     -19.608  -9.556 -13.987  1.00 31.03           C  \\nATOM    205  CD  GLN B   4     -19.344 -10.918 -13.379  1.00 54.72           C  \\nATOM    206  OE1 GLN B   4     -20.210 -11.794 -13.390  1.00 51.79           O  \\nATOM    207  NE2 GLN B   4     -18.144 -11.105 -12.840  1.00 41.70           N  \\nATOM    208  N   HIS B   5     -22.599  -8.080 -15.281  1.00 25.17           N  \\nATOM    209  CA  HIS B   5     -23.666  -7.471 -14.503  1.00 26.87           C  \\nATOM    210  C   HIS B   5     -23.358  -7.685 -13.035  1.00 26.33           C  \\nATOM    211  O   HIS B   5     -23.193  -8.822 -12.595  1.00 25.34           O  \\nATOM    212  CB  HIS B   5     -25.013  -8.109 -14.834  1.00 24.65           C  \\nATOM    213  CG  HIS B   5     -25.462  -7.871 -16.239  1.00 27.43           C  \\nATOM    214  ND1 HIS B   5     -26.235  -6.787 -16.598  1.00 29.87           N  \\nATOM    215  CD2 HIS B   5     -25.241  -8.570 -17.377  1.00 27.84           C  \\nATOM    216  CE1 HIS B   5     -26.478  -6.833 -17.896  1.00 24.52           C  \\nATOM    217  NE2 HIS B   5     -25.885  -7.905 -18.393  1.00 35.03           N  \\nATOM    218  N   LEU B   6     -23.267  -6.592 -12.287  1.00 22.69           N  \\nATOM    219  CA  LEU B   6     -22.974  -6.660 -10.862  1.00 21.06           C  \\nATOM    220  C   LEU B   6     -24.052  -5.932 -10.079  1.00 21.50           C  \\nATOM    221  O   LEU B   6     -24.197  -4.715 -10.194  1.00 22.66           O  \\nATOM    222  CB  LEU B   6     -21.612  -6.034 -10.568  1.00 20.56           C  \\nATOM    223  CG  LEU B   6     -20.418  -6.665 -11.281  1.00 20.16           C  \\nATOM    224  CD1 LEU B   6     -19.190  -5.786 -11.103  1.00 20.54           C  \\nATOM    225  CD2 LEU B   6     -20.155  -8.064 -10.763  1.00 25.22           C  \\nATOM    226  N   CYS B   7     -24.800  -6.679  -9.273  1.00 20.58           N  \\nATOM    227  CA  CYS B   7     -25.899  -6.104  -8.513  1.00 22.46           C  \\nATOM    228  C   CYS B   7     -25.754  -6.421  -7.034  1.00 22.72           C  \\nATOM    229  O   CYS B   7     -25.231  -7.471  -6.665  1.00 21.90           O  \\nATOM    230  CB  CYS B   7     -27.236  -6.664  -9.009  1.00 28.16           C  \\nATOM    231  SG  CYS B   7     -27.554  -6.459 -10.788  1.00 29.45           S  \\nATOM    232  N   GLY B   8     -26.229  -5.514  -6.191  1.00 21.36           N  \\nATOM    233  CA  GLY B   8     -26.233  -5.745  -4.759  1.00 22.98           C  \\nATOM    234  C   GLY B   8     -24.859  -6.051  -4.209  1.00 20.91           C  \\nATOM    235  O   GLY B   8     -23.911  -5.316  -4.465  1.00 23.25           O  \\nATOM    236  N   SER B   9     -24.741  -7.136  -3.447  1.00 18.12           N  \\nATOM    237  CA  SER B   9     -23.484  -7.444  -2.778  1.00 15.90           C  \\nATOM    238  C   SER B   9     -22.360  -7.702  -3.780  1.00 18.45           C  \\nATOM    239  O   SER B   9     -21.183  -7.509  -3.473  1.00 17.21           O  \\nATOM    240  CB  SER B   9     -23.648  -8.663  -1.863  1.00 19.34           C  \\nATOM    241  OG  SER B   9     -23.919  -9.813  -2.638  1.00 20.35           O  \\nATOM    242  N   HIS B  10     -22.730  -8.151  -4.972  1.00 14.91           N  \\nATOM    243  CA  HIS B  10     -21.764  -8.403  -6.039  1.00 17.86           C  \\nATOM    244  C   HIS B  10     -21.093  -7.097  -6.470  1.00 18.59           C  \\nATOM    245  O   HIS B  10     -19.891  -7.055  -6.725  1.00 17.96           O  \\nATOM    246  CB  HIS B  10     -22.471  -9.047  -7.236  1.00 18.56           C  \\nATOM    247  CG  HIS B  10     -23.081 -10.384  -6.933  1.00 23.08           C  \\nATOM    248  ND1 HIS B  10     -23.461 -11.269  -7.919  1.00 27.72           N  \\nATOM    249  CD2 HIS B  10     -23.366 -10.990  -5.755  1.00 23.69           C  \\nATOM    250  CE1 HIS B  10     -23.951 -12.364  -7.363  1.00 29.56           C  \\nATOM    251  NE2 HIS B  10     -23.906 -12.220  -6.050  1.00 22.03           N  \\nATOM    252  N   LEU B  11     -21.886  -6.037  -6.550  1.00 16.34           N  \\nATOM    253  CA  LEU B  11     -21.398  -4.713  -6.938  1.00 16.50           C  \\nATOM    254  C   LEU B  11     -20.523  -4.120  -5.847  1.00 19.44           C  \\nATOM    255  O   LEU B  11     -19.446  -3.585  -6.105  1.00 17.25           O  \\nATOM    256  CB  LEU B  11     -22.588  -3.788  -7.211  1.00 16.97           C  \\nATOM    257  CG  LEU B  11     -22.268  -2.390  -7.735  1.00 22.79           C  \\nATOM    258  CD1 LEU B  11     -21.211  -2.486  -8.812  1.00 17.51           C  \\nATOM    259  CD2 LEU B  11     -23.529  -1.754  -8.277  1.00 22.41           C  \\nATOM    260  N   VAL B  12     -20.990  -4.218  -4.610  1.00 15.40           N  \\nATOM    261  CA  VAL B  12     -20.242  -3.690  -3.490  1.00 13.81           C  \\nATOM    262  C   VAL B  12     -18.902  -4.422  -3.332  1.00 15.23           C  \\nATOM    263  O   VAL B  12     -17.879  -3.825  -2.988  1.00 14.98           O  \\nATOM    264  CB  VAL B  12     -21.096  -3.776  -2.212  1.00 24.43           C  \\nATOM    265  CG1 VAL B  12     -20.252  -3.609  -1.029  1.00 30.29           C  \\nATOM    266  CG2 VAL B  12     -22.169  -2.694  -2.241  1.00 21.29           C  \\nATOM    267  N   GLU B  13     -18.912  -5.721  -3.589  1.00 14.50           N  \\nATOM    268  CA  GLU B  13     -17.695  -6.515  -3.515  1.00 14.19           C  \\nATOM    269  C   GLU B  13     -16.682  -6.056  -4.574  1.00 15.30           C  \\nATOM    270  O   GLU B  13     -15.481  -5.936  -4.304  1.00 16.00           O  \\nATOM    271  CB  GLU B  13     -18.046  -7.986  -3.712  1.00 22.91           C  \\nATOM    272  CG  GLU B  13     -16.884  -8.875  -4.073  1.00 31.68           C  \\nATOM    273  CD  GLU B  13     -17.296 -10.336  -4.139  1.00 35.29           C  \\nATOM    274  OE1 GLU B  13     -17.726 -10.873  -3.092  1.00 24.65           O  \\nATOM    275  OE2 GLU B  13     -17.204 -10.932  -5.236  1.00 35.79           O  \\nATOM    276  N   ALA B  14     -17.166  -5.811  -5.782  1.00 16.94           N  \\nATOM    277  CA  ALA B  14     -16.298  -5.308  -6.856  1.00 19.02           C  \\nATOM    278  C   ALA B  14     -15.701  -3.941  -6.523  1.00 17.95           C  \\nATOM    279  O   ALA B  14     -14.507  -3.704  -6.731  1.00 16.40           O  \\nATOM    280  CB  ALA B  14     -17.065  -5.240  -8.157  1.00 19.13           C  \\nATOM    281  N   LEU B  15     -16.536  -3.036  -6.021  1.00 15.53           N  \\nATOM    282  CA  LEU B  15     -16.066  -1.724  -5.592  1.00 13.58           C  \\nATOM    283  C   LEU B  15     -14.977  -1.846  -4.540  1.00 20.33           C  \\nATOM    284  O   LEU B  15     -13.957  -1.159  -4.594  1.00 17.96           O  \\nATOM    285  CB  LEU B  15     -17.228  -0.890  -5.054  1.00 17.56           C  \\nATOM    286  CG  LEU B  15     -18.132  -0.253  -6.108  1.00 19.95           C  \\nATOM    287  CD1 LEU B  15     -19.434   0.217  -5.476  1.00 19.70           C  \\nATOM    288  CD2 LEU B  15     -17.420   0.908  -6.806  1.00 19.43           C  \\nATOM    289  N   TYR B  16     -15.196  -2.733  -3.580  1.00 16.41           N  \\nATOM    290  CA  TYR B  16     -14.214  -2.982  -2.528  1.00 14.22           C  \\nATOM    291  C   TYR B  16     -12.884  -3.467  -3.093  1.00 18.76           C  \\nATOM    292  O   TYR B  16     -11.823  -2.978  -2.708  1.00 14.92           O  \\nATOM    293  CB  TYR B  16     -14.766  -4.015  -1.551  1.00 13.58           C  \\nATOM    294  CG  TYR B  16     -13.748  -4.582  -0.595  1.00 15.85           C  \\nATOM    295  CD1 TYR B  16     -13.213  -3.811   0.431  1.00 19.91           C  \\nATOM    296  CD2 TYR B  16     -13.338  -5.909  -0.707  1.00 14.31           C  \\nATOM    297  CE1 TYR B  16     -12.290  -4.347   1.318  1.00 18.93           C  \\nATOM    298  CE2 TYR B  16     -12.419  -6.445   0.163  1.00 16.89           C  \\nATOM    299  CZ  TYR B  16     -11.899  -5.665   1.173  1.00 19.03           C  \\nATOM    300  OH  TYR B  16     -10.991  -6.225   2.037  1.00 19.26           O  \\nATOM    301  N   LEU B  17     -12.941  -4.437  -4.002  1.00 17.26           N  \\nATOM    302  CA  LEU B  17     -11.725  -4.998  -4.583  1.00 17.15           C  \\nATOM    303  C   LEU B  17     -10.983  -3.979  -5.439  1.00 20.48           C  \\nATOM    304  O   LEU B  17      -9.751  -3.889  -5.407  1.00 20.37           O  \\nATOM    305  CB  LEU B  17     -12.060  -6.221  -5.428  1.00 16.44           C  \\nATOM    306  CG  LEU B  17     -12.491  -7.468  -4.651  1.00 17.44           C  \\nATOM    307  CD1 LEU B  17     -12.867  -8.590  -5.593  1.00 22.58           C  \\nATOM    308  CD2 LEU B  17     -11.383  -7.898  -3.717  1.00 22.13           C  \\nATOM    309  N   VAL B  18     -11.738  -3.226  -6.224  1.00 18.15           N  \\nATOM    310  CA  VAL B  18     -11.150  -2.270  -7.151  1.00 18.50           C  \\nATOM    311  C   VAL B  18     -10.557  -1.063  -6.431  1.00 22.53           C  \\nATOM    312  O   VAL B  18      -9.454  -0.628  -6.747  1.00 21.63           O  \\nATOM    313  CB  VAL B  18     -12.188  -1.824  -8.207  1.00 19.51           C  \\nATOM    314  CG1 VAL B  18     -11.714  -0.594  -8.955  1.00 28.07           C  \\nATOM    315  CG2 VAL B  18     -12.468  -2.973  -9.165  1.00 23.98           C  \\nATOM    316  N   CYS B  19     -11.268  -0.548  -5.433  1.00 19.45           N  \\nATOM    317  CA  CYS B  19     -10.874   0.709  -4.803  1.00 21.28           C  \\nATOM    318  C   CYS B  19      -9.819   0.588  -3.702  1.00 26.60           C  \\nATOM    319  O   CYS B  19      -9.167   1.576  -3.355  1.00 25.82           O  \\nATOM    320  CB  CYS B  19     -12.102   1.455  -4.285  1.00 20.90           C  \\nATOM    321  SG  CYS B  19     -13.231   1.932  -5.593  1.00 21.21           S  \\nATOM    322  N   GLY B  20      -9.653  -0.608  -3.150  1.00 25.20           N  \\nATOM    323  CA  GLY B  20      -8.606  -0.825  -2.167  1.00 33.63           C  \\nATOM    324  C   GLY B  20      -8.661   0.162  -1.014  1.00 39.29           C  \\nATOM    325  O   GLY B  20      -9.737   0.464  -0.501  1.00 27.87           O  \\nATOM    326  N   GLU B  21      -7.507   0.678  -0.599  1.00 30.38           N  \\nATOM    327  CA  GLU B  21      -7.474   1.534   0.591  1.00 34.13           C  \\nATOM    328  C   GLU B  21      -8.014   2.944   0.351  1.00 23.63           C  \\nATOM    329  O   GLU B  21      -8.123   3.737   1.283  1.00 35.45           O  \\nATOM    330  CB  GLU B  21      -6.075   1.569   1.220  1.00 45.91           C  \\nATOM    331  CG  GLU B  21      -5.015   2.280   0.401  1.00 57.50           C  \\nATOM    332  CD  GLU B  21      -3.631   2.179   1.034  1.00 75.22           C  \\nATOM    333  OE1 GLU B  21      -2.707   2.876   0.563  1.00 79.46           O  \\nATOM    334  OE2 GLU B  21      -3.470   1.401   2.002  1.00 57.31           O  \\nATOM    335  N   ARG B  22      -8.356   3.251  -0.897  1.00 27.41           N  \\nATOM    336  CA  ARG B  22      -9.040   4.498  -1.213  1.00 28.81           C  \\nATOM    337  C   ARG B  22     -10.456   4.461  -0.645  1.00 33.76           C  \\nATOM    338  O   ARG B  22     -11.008   5.480  -0.235  1.00 31.35           O  \\nATOM    339  CB  ARG B  22      -9.130   4.700  -2.726  1.00 30.48           C  \\nATOM    340  CG  ARG B  22      -7.876   5.228  -3.400  1.00 38.41           C  \\nATOM    341  CD  ARG B  22      -8.072   5.288  -4.922  1.00 44.34           C  \\nATOM    342  NE  ARG B  22      -8.003   3.963  -5.546  1.00 61.53           N  \\nATOM    343  CZ  ARG B  22      -8.246   3.711  -6.833  1.00 56.00           C  \\nATOM    344  NH1 ARG B  22      -8.151   2.467  -7.288  1.00 37.21           N  \\nATOM    345  NH2 ARG B  22      -8.588   4.690  -7.664  1.00 33.73           N  \\nATOM    346  N   GLY B  23     -11.048   3.276  -0.627  1.00 23.72           N  \\nATOM    347  CA  GLY B  23     -12.442   3.151  -0.243  1.00 25.22           C  \\nATOM    348  C   GLY B  23     -13.323   3.633  -1.376  1.00 26.84           C  \\nATOM    349  O   GLY B  23     -12.832   3.954  -2.460  1.00 20.71           O  \\nATOM    350  N   PHE B  24     -14.625   3.700  -1.132  1.00 18.98           N  \\nATOM    351  CA  PHE B  24     -15.558   4.055  -2.188  1.00 17.55           C  \\nATOM    352  C   PHE B  24     -16.875   4.561  -1.614  1.00 23.50           C  \\nATOM    353  O   PHE B  24     -17.092   4.530  -0.402  1.00 24.02           O  \\nATOM    354  CB  PHE B  24     -15.811   2.837  -3.086  1.00 17.36           C  \\nATOM    355  CG  PHE B  24     -16.428   1.661  -2.361  1.00 19.89           C  \\nATOM    356  CD1 PHE B  24     -15.628   0.691  -1.776  1.00 18.48           C  \\nATOM    357  CD2 PHE B  24     -17.811   1.527  -2.269  1.00 22.89           C  \\nATOM    358  CE1 PHE B  24     -16.189  -0.390  -1.107  1.00 21.90           C  \\nATOM    359  CE2 PHE B  24     -18.378   0.446  -1.597  1.00 19.48           C  \\nATOM    360  CZ  PHE B  24     -17.562  -0.513  -1.015  1.00 19.73           C  \\nATOM    361  N   PHE B  25     -17.742   5.048  -2.487  1.00 23.02           N  \\nATOM    362  CA  PHE B  25     -19.108   5.343  -2.088  1.00 26.05           C  \\nATOM    363  C   PHE B  25     -20.081   4.558  -2.948  1.00 29.87           C  \\nATOM    364  O   PHE B  25     -19.912   4.448  -4.158  1.00 25.71           O  \\nATOM    365  CB  PHE B  25     -19.410   6.848  -2.118  1.00 30.50           C  \\nATOM    366  CG  PHE B  25     -19.035   7.523  -3.399  1.00 36.46           C  \\nATOM    367  CD1 PHE B  25     -17.729   7.929  -3.625  1.00 41.85           C  \\nATOM    368  CD2 PHE B  25     -19.989   7.772  -4.374  1.00 47.50           C  \\nATOM    369  CE1 PHE B  25     -17.376   8.562  -4.808  1.00 51.84           C  \\nATOM    370  CE2 PHE B  25     -19.645   8.407  -5.560  1.00 37.96           C  \\nATOM    371  CZ  PHE B  25     -18.336   8.802  -5.778  1.00 47.77           C  \\nATOM    372  N   TYR B  26     -21.090   3.991  -2.301  1.00 26.11           N  \\nATOM    373  CA  TYR B  26     -22.087   3.184  -2.976  1.00 19.37           C  \\nATOM    374  C   TYR B  26     -23.429   3.866  -2.826  1.00 26.37           C  \\nATOM    375  O   TYR B  26     -23.966   3.954  -1.726  1.00 25.08           O  \\nATOM    376  CB  TYR B  26     -22.136   1.795  -2.350  1.00 18.99           C  \\nATOM    377  CG  TYR B  26     -23.255   0.930  -2.876  1.00 21.56           C  \\nATOM    378  CD1 TYR B  26     -24.280   0.506  -2.042  1.00 24.62           C  \\nATOM    379  CD2 TYR B  26     -23.285   0.533  -4.203  1.00 23.44           C  \\nATOM    380  CE1 TYR B  26     -25.304  -0.295  -2.516  1.00 26.30           C  \\nATOM    381  CE2 TYR B  26     -24.304  -0.265  -4.690  1.00 25.09           C  \\nATOM    382  CZ  TYR B  26     -25.313  -0.675  -3.842  1.00 27.86           C  \\nATOM    383  OH  TYR B  26     -26.328  -1.469  -4.320  1.00 29.25           O  \\nATOM    384  N   THR B  27     -23.960   4.365  -3.936  1.00 28.14           N  \\nATOM    385  CA  THR B  27     -25.185   5.147  -3.909  1.00 37.14           C  \\nATOM    386  C   THR B  27     -26.169   4.594  -4.936  1.00 32.53           C  \\nATOM    387  O   THR B  27     -26.193   5.035  -6.082  1.00 36.20           O  \\nATOM    388  CB  THR B  27     -24.889   6.637  -4.188  1.00 36.89           C  \\nATOM    389  OG1 THR B  27     -24.271   6.772  -5.473  1.00 57.47           O  \\nATOM    390  CG2 THR B  27     -23.936   7.182  -3.151  1.00 33.01           C  \\nATOM    391  N   PRO B  28     -26.989   3.620  -4.517  1.00 29.25           N  \\nATOM    392  CA  PRO B  28     -27.893   2.887  -5.411  1.00 32.94           C  \\nATOM    393  C   PRO B  28     -29.081   3.726  -5.872  1.00 32.76           C  \\nATOM    394  O   PRO B  28     -29.671   3.417  -6.908  1.00 36.05           O  \\nATOM    395  CB  PRO B  28     -28.401   1.730  -4.542  1.00 31.48           C  \\nATOM    396  CG  PRO B  28     -27.658   1.813  -3.247  1.00 28.54           C  \\nATOM    397  CD  PRO B  28     -27.128   3.190  -3.118  1.00 34.39           C  \\nATOM    398  N   LYS B  29     -29.436   4.751  -5.105  1.00 34.79           N  \\nATOM    399  CA  LYS B  29     -30.532   5.642  -5.483  1.00 49.07           C  \\nATOM    400  C   LYS B  29     -30.051   6.834  -6.309  1.00 63.70           C  \\nATOM    401  O   LYS B  29     -30.669   7.191  -7.315  1.00 73.47           O  \\nATOM    402  CB  LYS B  29     -31.305   6.122  -4.251  1.00 62.86           C  \\nATOM    403  CG  LYS B  29     -32.706   5.526  -4.105  1.00 74.15           C  \\nATOM    404  CD  LYS B  29     -32.690   4.118  -3.525  1.00 72.32           C  \\nATOM    405  CE  LYS B  29     -32.356   3.061  -4.572  1.00 68.55           C  \\nATOM    406  NZ  LYS B  29     -33.458   2.854  -5.553  1.00 59.40           N  \\nATOM    407  N   ALA B  30     -28.950   7.446  -5.884  1.00 72.05           N  \\nATOM    408  CA  ALA B  30     -28.366   8.568  -6.616  1.00 74.07           C  \\nATOM    409  C   ALA B  30     -27.782   8.115  -7.955  1.00 73.01           C  \\nATOM    410  O   ALA B  30     -27.485   8.928  -8.831  1.00 64.47           O  \\nATOM    411  CB  ALA B  30     -27.302   9.269  -5.772  1.00 56.14           C  \\nATOM    412  OXT ALA B  30     -27.594   6.921  -8.197  1.00 73.19           O  \\nTER     413      ALA B  30                                                      \\nHETATM  414  O   HOH A  22     -29.500  -7.696 -14.149  1.00 50.68           O  \\nHETATM  415  O   HOH A  23     -11.374  10.214 -16.391  1.00 40.62           O  \\nHETATM  416  O   HOH A  24     -11.187   9.560 -19.007  1.00 44.09           O  \\nHETATM  417  O   HOH A  25     -22.708  -0.422 -18.245  1.00 25.03           O  \\nHETATM  418  O   HOH A  26     -17.282   8.923   0.122  1.00 44.14           O  \\nHETATM  419  O   HOH A  27     -27.802  -5.289 -20.151  1.00 41.02           O  \\nHETATM  420  O   HOH A  28     -13.732   9.674 -11.684  1.00 48.35           O  \\nHETATM  421  O   HOH A  29     -26.166   6.685 -14.912  1.00 50.84           O  \\nHETATM  422  O   HOH A  30     -30.826  -5.812 -17.868  1.00 54.11           O  \\nHETATM  423  O   HOH A  31     -14.803   9.989  -8.715  1.00 62.79           O  \\nHETATM  424  O   HOH A  32     -19.609   9.964  -9.626  1.00 46.30           O  \\nHETATM  425  O   HOH A  33     -13.690  11.749 -14.300  1.00 49.90           O  \\nHETATM  426  O   HOH B  31     -24.729  10.263 -12.014  1.00 61.01           O  \\nHETATM  427  O   HOH B  32     -20.724 -10.273 -19.319  1.00 52.72           O  \\nHETATM  428  O   HOH B  33     -17.700 -13.422  -2.674  1.00 25.54           O  \\nHETATM  429  O   HOH B  34     -28.494  10.301 -14.536  1.00 56.88           O  \\nHETATM  430  O   HOH B  35     -24.545   6.027  -8.162  1.00 32.84           O  \\nHETATM  431  O   HOH B  36     -27.182  -2.924  -7.024  1.00 27.65           O  \\nHETATM  432  O   HOH B  37     -24.782  -9.599  -9.935  1.00 26.60           O  \\nHETATM  433  O   HOH B  38     -14.505  -9.421 -15.869  1.00 44.83           O  \\nHETATM  434  O   HOH B  39     -28.442  -1.829  -2.831  1.00 43.18           O  \\nHETATM  435  O   HOH B  40     -12.046  -0.492  -0.930  1.00 25.29           O  \\nHETATM  436  O   HOH B  41     -18.702  -9.557  -7.601  1.00 29.84           O  \\nHETATM  437  O   HOH B  42      -8.200  -0.632  -9.134  1.00 32.65           O  \\nHETATM  438  O   HOH B  43     -15.277 -10.853  -7.465  1.00 43.14           O  \\nHETATM  439  O   HOH B  44     -22.107   4.402  -6.065  1.00 31.09           O  \\nHETATM  440  O   HOH B  45      -5.206   0.010  -2.235  1.00 43.01           O  \\nHETATM  441  O   HOH B  46     -26.729  -9.715  -6.146  1.00 30.80           O  \\nHETATM  442  O   HOH B  47     -27.875   3.972  -9.042  1.00 40.15           O  \\nHETATM  443  O   HOH B  48     -20.453  -7.642  -0.550  1.00 34.77           O  \\nHETATM  444  O   HOH B  49     -17.760 -10.041  -0.552  1.00 37.12           O  \\nHETATM  445  O   HOH B  50     -22.932 -10.505 -16.556  1.00 47.78           O  \\nHETATM  446  O   HOH B  51     -28.195   5.830  -2.571  1.00 53.39           O  \\nHETATM  447  O   HOH B  52     -17.118 -11.123 -18.586  1.00 52.94           O  \\nHETATM  448  O   HOH B  53     -27.581   8.994 -11.659  1.00 53.89           O  \\nCONECT   43   76                                                                \\nCONECT   49  231                                                                \\nCONECT   76   43                                                                \\nCONECT  162  321                                                                \\nCONECT  231   49                                                                \\nCONECT  321  162                                                                \\nMASTER      278    0    0    4    0    0    0    6  438    2    6    5          \\nEND                                                                             \\n\",\"pdb\");\n",
       "\tviewer_17563213462829392.setStyle({\"cartoon\": {\"color\": \"spectrum\"}});\n",
       "\tviewer_17563213462829392.zoomTo();\n",
       "viewer_17563213462829392.render();\n",
       "});\n",
       "</script>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "import py3Dmol\n",
    "import requests\n",
    "import numpy as np\n",
    "\n",
    "\n",
    "def fetch_protein_structure(pdb_id) -> str:\n",
    "    url = f\"https://files.rcsb.org/download/{pdb_id}.pdb\"\n",
    "    response = requests.get(url)\n",
    "    if response.status_code == 200:\n",
    "        return response.text\n",
    "    else:\n",
    "        raise ValueError(f\"Could not fetch PDB file for ID {pdb_id}\")\n",
    "    \n",
    "protein_to_pdb = {\n",
    "    \"insulin\": \"3I40\",\n",
    "    \"collagen\": \"1BKV\",\n",
    "    \"proteasome\": \"1YAR\",\n",
    "}\n",
    "\n",
    "protein = \"insulin\"\n",
    "pdb_structure = fetch_protein_structure(pdb_id=protein_to_pdb[protein])\n",
    "\n",
    "pdbview = py3Dmol.view(width=400, height=300)\n",
    "pdbview.addModel(pdb_structure, \"pdb\")\n",
    "pdbview.setStyle({\"cartoon\": {\"color\": \"spectrum\"}})\n",
    "pdbview.zoomTo()\n",
    "pdbview.show()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "c8a7f67c",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'R': 0, 'N': 1, 'D': 2, 'C': 3, 'Q': 4, 'E': 5, 'G': 6, 'H': 7, 'I': 8, 'L': 9}\n",
      "(+10 more entries)\n"
     ]
    }
   ],
   "source": [
    "from dlfb.utils.display import print_short_dict\n",
    "\n",
    "amino_acids = [ \"R\", \"N\", \"D\", \"C\", \"Q\", \"E\", \"G\", \"H\", \"I\", \"L\", \"K\", \"M\", \"F\", \"P\", \"S\", \"T\", \"W\", \"Y\", \"V\", \"A\" ]\n",
    "\n",
    "amino_acid_to_index = {\n",
    "    amino_acid: index for index, amino_acid in enumerate(amino_acids)\n",
    "}\n",
    "\n",
    "print_short_dict(amino_acid_to_index)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "d0206645",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[11, 5, 15, 19, 14, 15, 19, 14, 5]"
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "tiny_protein = [\"M\", \"E\", \"T\", \"A\", \"S\", \"T\", \"A\", \"S\", \"E\" ]\n",
    "\n",
    "tiny_protein_indices = [\n",
    "    amino_acid_to_index[amino_acid] for amino_acid in tiny_protein  \n",
    "]\n",
    "\n",
    "tiny_protein_indices"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "09454ca8",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO:2025-08-27 14:08:18,700:jax._src.xla_bridge:830: Unable to initialize backend 'tpu': INTERNAL: Failed to open libtpu.so: libtpu.so: cannot open shared object file: No such file or directory\n",
      "2025-08-27 14:08:18.700 INFO jax._src.xla_bridge xla_bridge - backends: Unable to initialize backend 'tpu': INTERNAL: Failed to open libtpu.so: libtpu.so: cannot open shared object file: No such file or directory\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[[0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 1. 0. 0. 0. 0. 0. 0. 0. 0.]\n",
      " [0. 0. 0. 0. 0. 1. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0.]\n",
      " [0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 1. 0. 0. 0. 0.]\n",
      " [0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 1.]\n",
      " [0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 1. 0. 0. 0. 0. 0.]\n",
      " [0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 1. 0. 0. 0. 0.]\n",
      " [0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 1.]\n",
      " [0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 1. 0. 0. 0. 0. 0.]\n",
      " [0. 0. 0. 0. 0. 1. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0.]]\n"
     ]
    }
   ],
   "source": [
    "import jax\n",
    "\n",
    "one_hot_encoded_sequence = jax.nn.one_hot(\n",
    "    x=tiny_protein_indices, num_classes=len(amino_acids)\n",
    ")\n",
    "\n",
    "print(one_hot_encoded_sequence)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "id": "968b59ec",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Text(0.5, 96.72222222222221, 'Protein sequence')"
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAi8AAAEeCAYAAABCGvXRAAAAOnRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjEwLjUsIGh0dHBzOi8vbWF0cGxvdGxpYi5vcmcvWftoOwAAAAlwSFlzAAAPYQAAD2EBqD+naQAAMeJJREFUeJzt3XlYVGX/BvD7sDgg4igiMMjiLuJCIpaoiRkp1s9wKUxNMTRTUcEVUwvNFKzcylfLBTWtzDLN3LGQJEMFxEgJMVHRINxXGJE5vz98nddhcwbOCGe4P9c116VnZr7f51hHb57znHMEURRFEBEREcmEWVUPgIiIiMgQDC9EREQkKwwvREREJCsML0RERCQrDC9EREQkKwwvREREJCsML0RERCQrDC9EREQkKwwvREREJCsWVT0AYxAEy6oeAhHJ2ANxvdF7WAgjjN6DSG5EsVCvz3HmhYiIiGSF4YWIiIhkheGFiIiIZIXhhYiIiGSF4YWIiIhkheGFiIiIZIXhhYiIiGSF4YWIiIhkheGFiIiIZKVK77B78eJFrFy5EocPH0Zubi4EQYCjoyO6dOmCMWPGwNXVtSqHR0RERNWQIIqiWBWNExIS0KdPH7i6uqJXr15wdHSEKIrIy8tDbGwssrOzsWfPHnTt2rXcOmq1Gmq1WmebUmkHQDDi6InIlPHxAERVQ9/HA1RZeOnUqRO6deuGJUuWlPr+pEmTkJCQgGPHjpVbZ86cOZg7d26xrQIAc2kGSkQ1DsMLUdWo9uHF2toaqampaNWqVanv//XXX+jQoQPy8/PLrcOZFyKSGsMLUdXQN7xU2ZoXlUqFw4cPlxlefv/9d6hUqifWUSgUUCgUxbYyuBAREZmqKgsvU6dOxZgxY5CcnIyXXnoJjo6OEAQBubm5iI2NxZo1a7B06dKqGh4RERFVU1UWXsaNG4cGDRpgyZIl+OKLL1BUVAQAMDc3R8eOHfHll18iKCioqoZHRERE1VSVrXl5XGFhIa5cuQIAsLe3h6WlZaXqCULlvk9ENRvXvBBVjWq/5uVxlpaWeq1vISIiIuIddomIiEhWGF6IiIhIVhheiIiISFYYXoiIiEhWGF6IiIhIVqrFpdJS46XSNQsvayUiMg36XirNmRciIiKSFYYXIiIikhWGFyIiIpIVhhciIiKSFYYXIiIikhWGFyIiIpIVhhciIiKSFYYXIiIikhWGFyIiIpIVhhciIiKSFYYXIiIikpVqHV6ys7MREhJS7mfUajVu3bql8wJM7nFNRERE9F/VOrxcu3YNGzZsKPczUVFRUCqVOi9A83QGSERERE9dlT5VeseOHeW+f/bsWUyZMgVFRUVlfkatVkOtVutsUyrtAAhSDJFkgE+VJiIyDfo+VbpKw4uZmRkEQUB5QxAEodzwUvp3LCs7NJIRhhciItOgb3ip0tNGKpUKW7duhUajKfWVkpJSlcMjIiKiaqhKw0vHjh3LDShPmpUhIiKimseiKptPmzYNd+/eLfP95s2bIy4u7imOiIiIiKq7Kl3zYixc81KzcM0LEZFpkMWaFyIiIiJDMbwQERGRrDC8EBERkawwvBAREZGsMLwQERGRrPBqIyIiqrZ4NWHNwquNiIiIyCQxvBAREZGsSBpeTPAMFBEREVUzBoeXYcOG4c6dOyW2nzt3Dt27d5dkUERERERlMTi8nDp1Cu3atcNvv/2m3bZhwwZ4eXnB0dFR0sERERERFWfwgxmPHDmC2bNno2fPnpgyZQoyMzOxd+9eLFu2DCEhIcYYIxEREZGWweHFwsIC0dHRUCgUmDdvHiwsLBAfHw9fX19jjI+IiIhIh8GnjQoLCzFlyhQsXLgQ7777Lnx9fdG/f3/s3r3bGOMjIiIi0mHwzIuPjw/u3buHgwcPonPnzhBFER999BEGDBiAkJAQrFixwhjjJCIiIgJQgZkXHx8fpKamonPnzgAAQRAQERGBxMRE/Prrr5IPkIiIiOhxkj4eQK1WQ6FQGPSd/Px8JCcnw87ODp6enjrvFRQUYMuWLRg+fLhBNfl4ACIi08DHA9QsRn08wMaNG9G1a1c4Ozvj/PnzAIClS5di7969BtU5ffo0Wrduje7du6Ndu3bo0aMHcnJytO/fvHkTb731Vrk11Go1bt26pfMCeLM8IiIiU2VweFm5ciUmT56Ml19+GTdu3EBRUREAoF69eli6dKlBtSIiItCuXTvk5eUhIyMDdevWRdeuXXHhwgW9a0RFRUGpVOq8AI1B4yAiIiL5MPi0kaenJxYsWIB+/frB1tYWJ06cQNOmTfHnn3+iR48euHLlit61HB0dceDAAbRr1067LTQ0FDt37kRcXBxsbGzg7OysDUilUavVUKvVOtuUSjsAgiG7RURE1RBPG9Us+p42Mvhqo6ysLHTo0KHEdoVCgbt37xpUKz8/HxYWukP4z3/+AzMzM/j5+eHrr79+Yg2FQlHKOhsGFyIiIlNlcHhp0qQJUlNT4e7urrN9z549JRbcPomHhweSkpLQunVrne2fffYZRFHEq6++aujwiIiIyMQZHF6mTZuG0NBQFBQUQBRFHD16FN988w2ioqKwZs0ag2r1798f33zzDYYNG1biveXLl0Oj0eDzzz83dIhERERkwip0qfTq1avx4YcfIjs7GwDQqFEjzJkzByNHjpR8gBXBS6WJiEwD17zULPqueanUfV6uXLkCjUYDBweHipYwCoYXIiLTwPBSsxhtwe7j7O3tK/N1IiIiIoPpFV46dOgAQdDvCp6UlJRKDYiIiIioPHqFl379+ml/XVBQgBUrVsDT0xO+vr4AgMTERJw8eRLjxo0zyiCJiIiIHjF4zcuoUaOgUqkwb948ne2RkZHIzs5GTEyMpAOsCK55ISIyDVzzUrMYbcGuUqlEUlISWrRoobM9MzMTPj4+uHnzpiHljILhhYjINDC81CxGW7BrbW2NhISEEuElISEBVlZWhpYjIiIqE4NF9fE0gqS+DA4v4eHhGDt2LJKTk9G5c2cAD9e8xMTE4P3335d8gERERESPq9B9XrZs2YJly5YhPT0dANC6dWuEhYUhKChI8gFWBE8bERERSetpzLyYY6hen6vUTeqqK4YXIiIiaVWn8FLhm9Tdv38feXl50Gg0Otvd3NwqWpKIiIjoiQwOL5mZmQgJCcHhw4d1touiCEEQUFRUJNngiIiIiIozOLyMGDECFhYW2LlzJ1Qqld533iUiIiKSgsHhJTU1FcnJyfDw8DDGeIiIiIjKZWboFzw9PXHlyhVjjIWIiIjoiQwOLwsXLsT06dNx8OBBXL16Fbdu3dJ5ERERERmTwZdKm5k9zDvF17pUpwW7vFSaiIhIWrK+VDouLs7gwZQnPT0diYmJ8PX1hYeHB/766y8sW7YMarUab775Jnr27Fnu99VqNdRqdbGtIgAuJCYiIjJFBocXPz8/yZrv3bsXgYGBqFOnDu7du4dt27Zh+PDh8PLygiiK6N27N/bt21dugImKisLcuXOLbRUAmEs2TiIiIqo+9D5t9Mcff+hVsH379no379KlC3r27IkPP/wQmzdvxrhx4zB27FjMnz8fADBr1iwcO3YM+/fvL7NGaTMvSqUdOPNCREQknep02kjv8GJmZgZBEFDexw1d86JUKpGcnIzmzZtDo9FAoVDgyJEj8Pb2BgD8+eef8Pf3R25urt41H46Da16IiIikVJ3Ci96njbKysio8GH2YmZnBysoK9erV026ztbXFzZs3jdqXiIiI5EXv8OLu7i5588aNG+PMmTNo3rw5AOD333/XeTZSdnY2VCqV5H2JiIhIvir8YEYpjB07Vuc0U9u2bXXe37NnzxOvNiIiIqKaxeD7vMgB17wQERFJqzqteTH4DrtEREREVYnhhYiIiGSF4YWIiIhkRa8Fux06dCjxLKOypKSkVGpAREREROXRK7z069dP++uCggKsWLECnp6e8PX1BQAkJibi5MmTGDdunFEGSURERPSIwVcbjRo1CiqVCvPmzdPZHhkZiezsbMTExEg6wIrg1UZERETSqk5XGxkcXpRKJZKSktCiRQud7ZmZmfDx8akWd8RleCEiMr6n8Y+ZhTDC6D2o+hDFQr0+Z/CCXWtrayQkJJTYnpCQACsrK0PLERERERnE4DvshoeHY+zYsUhOTkbnzp0BPFzzEhMTg/fff1/yARIRERE9rkJ32N2yZQuWLVuG9PR0AEDr1q0RFhaGoKAgyQdYETxtRERkfDxtRFLT97QRHw9AREQVwvBCUjPamhciIiKiqqTXmhc7OzucPn0a9vb2qF+/frk3rLt27ZpkgyMiIiIqTq/wsmTJEtja2mp/re/ddomIiIikxjUvRERUIVzzQlIz2pqX3bt3Y9++fSW279+/H3v27DG0XAkmmKWIiIhIQgaHlxkzZqCoqKjEdo1GgxkzZlR6QAqFQnsJNhEREVFxBt+kLjMzE56eniW2e3h44MyZM3rXmTx5cqnbi4qKEB0djQYNGgAAFi9eXG4dtVoNtVpdbKsIgOtyiIiITJHB4UWpVOLs2bNo3LixzvYzZ87AxsZG7zpLly6Fl5cX6tWrp7NdFEWkp6fDxsZGr4XBUVFRmDt3brGtAgBzvcdCRERE8mHwgt3Ro0cjMTER27ZtQ7NmzQA8DC4DBw5Ep06dsGbNGr3qREVFYfXq1VizZg169uyp3W5paYkTJ06UOrtTmtJmXpRKO3DmhYjIuLhgl6RmtDvs3rx5EwEBAUhKSoKLiwsA4OLFi3j++eexdetW1K9fX+9ax44dw5tvvom+ffsiKioKlpaWBoeX0vBqIyIi42N4IanpG14qdNro8OHDiI2NxYkTJ2BtbY327duje/fuBg+yU6dOSE5ORmhoKHx8fLBp0ybeQ4aIiIjKJcl9XjQaDXbt2oW1a9di+/btFaqxefNmhIeH4/Lly0hLS+PMCxFRNceZF5LaU3m2UWZmJt599124uLhU+onSb7zxBpKSkvDDDz/A3d29UrWIiIjIdBl82ig/Px9btmzB2rVrkZiYiKKiIixZsgQhISGoU6dOpQbj4uKiXUdDREREVBq9Z16OHj2K0aNHw8nJCcuXL8fAgQORnZ0NMzMz+Pv7Vzq4EBEREelD75mXLl26YMKECTh69ChatWplzDERERERlUnv8NKzZ0+sXbsWeXl5GDZsGHr37s0rg4iIiOip0/u00f79+3Hy5Em0atUKY8eOhUqlQlhYGAAwxBAREdFTY9DVRq6urnj//feRlZWFjRs3Ii8vDxYWFggMDMTMmTORkpJirHESERERAZDgPi/Xr1/Hpk2bEBMTgz/++KPUJ04/bbzPCxGR8fE+LyQ1oz0eoDwpKSnw9vaWqlyFMbwQEZkGBqSa5ancpK646hBciIiIyLRJGl6IiIiIjI3hhYiIiGSF4YWIiIhkxeBnGz1y+fJlZGRkQBAEtGzZEg0bNpRyXERERESlMnjm5e7duwgJCYGzszO6d++O559/Hs7Ozhg5ciTu3btnjDESERERaRkcXiZPnoz4+Hjs2LEDN27cwI0bN/Djjz8iPj4eU6ZMMcYYiYiIiLQMvs+Lvb09vv/+e/To0UNne1xcHIKCgnD58mUpx1chvM8LEZFp4H1eahaj3efl3r17cHR0LLHdwcGBp42IiIjI6AwOL76+voiMjERBQYF2W35+PubOnQtfX19JB0dERERUnMFXGy1btgwBAQFwcXGBl5cXBEFAamoqrKyssG/fvkoN5vr169iwYQMyMzOhUqkQHBwMV1fXcr+jVquhVquLbRUB8EnXREREpqhCzzbKz8/Hpk2b8Ndff0EURXh6emLo0KGwtrY2qI6zszPS0tLQoEEDZGVloUuXLgCAdu3aIT09Hbdv30ZiYiI8PDzKrDFnzhzMnTu3+G4BMDdwr4iIqLrhmpeapUoezGgoMzMz5ObmwsHBAYMHD0Zubi527dqF2rVrQ61W47XXXoOVlRW+++67MmuUNvOiVNqBMy9ERPLH8FKz6BteKnSTutOnT+PgwYPIy8uDRqPRee/999+vSEkcOXIEa9asQe3atQEACoUCs2fPxmuvvVbu9xQKBRQKRbGtDC5ERESmyuDwsnr1aowdOxb29vZwcnKCIPwvKAiCYHB4efR9tVpd4iomR0fHanHpNREREVUfBoeXDz/8EPPnz0dERIQkA3jxxRdhYWGBW7du4fTp02jTpo32vQsXLsDe3l6SPkRERGQaDA4v169fx+uvvy5J88jISJ3fPzpl9MhPP/2E559/XpJeREREZBoMXrA7cuRIdOrUCWPGjDHWmCqNd9glIjINXLBbsxhtwW7z5s3x3nvvITExEe3atYOlpW5QmDhxoqEliYiIiPRm8MxLkyZNyi4mCDh79mylB1VZnHkhIjINnHmpWYw285KVlWXwYIiIiIikYvCzjYiIiIiqkl4zL5MnT8a8efNgY2ODyZMnl/vZxYsXSzIwIiIiotLoFV6OHz+OwsJC7a/L8vgN64iIiCqL61Gqj6ex/khfVfpsI2Phgl0iIiJpPY3wYo6hen2Oa16IiIhIVgy+2qigoACfffYZ4uLiSn0wY0pKimSDIyIiIirO4PASEhKC2NhYvPbaa3j22We5zoWIiIieKoPDy65du7B792507drVGOMhIiIiKpfBa14aNWoEW1tbY4yFiIiI6IkMDi+LFi1CREQEzp8/b4zxEBEREZXL4NNGPj4+KCgoQNOmTVG7du0SD2a8du2aZIMjIiIiKs7g8DJ48GBcunQJCxYsgKOjIxfsEhER0VNlcHg5fPgwfv/9d3h5eRljPERERETlMnjNi4eHB/Lz840xFiIiIqInMji8REdHY8qUKTh48CCuXr2KW7du6bwMcfz4cWRlZWl/v2nTJnTt2hWurq7o1q0bNm/e/MQaarW6lDGY3BMPiIiI6L8MDi8BAQH4/fff8eKLL8LBwQH169dH/fr1Ua9ePdSvX9+gWiNHjsS5c+cAAGvWrMHo0aPh4+ODWbNmoVOnTnj77bcRExNTbo2oqCgolUqdF6Ap9ztEREQkXwY/mDE+Pr7c9/38/PSuZWNjg/T0dLi5ucHb2xtjxozB6NGjte9//fXXmD9/Pk6ePFlmDbVaDbVarbNNqbQDwIXEREREUqlOD2Y0eMFueeEkNTXVoFrW1ta4fPky3NzccOnSJTz33HM67z/33HM6p5VKo1AooFAoim1lcCEiIjJVlX6q9M2bN7FixQp4e3ujY8eOBn23T58+WLlyJYCHoej777/XeX/Lli1o3rx5ZYdIREREJsTgmZdHfvnlF8TExOCHH36Au7s7Bg4ciLVr1xpUY+HChejatSv8/Pzg4+ODRYsW4eDBg2jdujUyMjKQmJiIbdu2VXSIREREZIIMCi8XL17E+vXrERMTg7t37yIoKAiFhYXYunUrPD09DW7u7OyM48ePIzo6Gj/99BNEUcTRo0eRnZ2Nrl274rfffoOPj4/BdYmIiMh06b1g9+WXX0ZCQgL+7//+D0OHDkVAQADMzc1haWmJEydOVCi8GIsgWD75Q0RERKQ3WS7Y3b9/PyZOnIixY8eiRYsWFR4YERERUWXovWD30KFDuH37Nnx8fPDcc89h+fLluHz5sjHHRkRERFSC3uHF19cXq1evRk5ODt555x1s3rwZjRo1gkajQWxsLG7fvm3McRIREREBqMBN6h6XkZGBtWvXYuPGjbhx4wZeeukl7NixQ8rxVQjXvBAREUmrOq15qdR9Xlq1aoWPPvoIFy9exDfffFOZUkRERER6qdTMS3XFmRciIuN7Gj+JWwgjjN6Dqg9RLNTrc5W+wy4RERHR08TwQkRERLLC8EJERESywvBCREREssLwQkRERLLC8EJERESywvBCREREssLwQkRERLLC8EJERESywvBCREREslKl4WXChAk4dOhQVQ6BiIiIZKZKn21kZmYGQRDQrFkzjBw5EsHBwXBycjKohlqthlqt1tmmVNoBECQcKRERFcdnG5HUZPNso/379+Pll1/GJ598Ajc3NwQGBmLnzp3QaDR6fT8qKgpKpVLnBej3XSIiIpKfKp95yc3NhYODAwoLC7Ft2zbExMTgwIEDcHR0xIgRI/DWW2+hefPmZdbgzAsRUdXgzAtJTd+Zl2oTXh534cIFxMTEYP369cjOzkZRUZFBdQXBUsphEhFRKRheSGqyDi+PiKKIAwcO4KWXXjKoLsMLEZHxMbyQ1GSx5sXd3R3m5uZlvi8IgsHBhYiIiEybRVU2z8rKqsr2REREJENVfrURERERkSEYXoiIiEhWGF6IiIhIVhheiIiISFYYXoiIiEhWGF6IiIhIVqr0JnXGwpvU1Sy8URYRkWmQxU3qiIiIiAzF8EJERESywvBCREREssLwQkRERLLC8EJERESywvBCREREssLwQkRERLLC8EJERESywvBCREREssLwQkRERLJS5eHls88+Q3BwMLZs2QIA2LhxIzw9PeHh4YGZM2fiwYMH5X5frVbj1q1bOi/A5J54QERERP9lUZXN582bh48//hi9evVCWFgYsrKy8PHHH2PSpEkwMzPDkiVLYGlpiblz55ZZIyoqqpT3BQDmRh07ERERVY0qfTBjs2bN8PHHH2PAgAE4ceIEOnbsiA0bNmDo0KEAgG3btmH69OnIzMwss4ZarYZardbZplTa4WGAoZqAD2YkIjIN+j6YsUpnXnJycuDj4wMA8PLygpmZGZ555hnt+97e3vjnn3/KraFQKKBQKIptZXAhIiIyVVW65sXJyQmnTp0CAGRmZqKoqEj7ewA4efIkHBwcqmp4REREVA1V6czLkCFDMHz4cAQGBuLnn39GREQEpk6diqtXr0IQBMyfPx+vvfZaVQ6RiIiIqpkqDS9z586FtbU1EhMT8c477yAiIgLt27fH9OnTce/ePfTt2xfz5s2ryiESERFRNVOlC3aNRRAsq3oI9BRxwS4RkWnQd8Fuld/nhYiIiMgQDC9EREQkKwwvREREJCsML0RERCQrDC9EREQkKwwvREREJC8iiQUFBWJkZKRYUFAgy/rsUb16mMI+sEf1qc8e1auHKeyDKfQwyfu8GOrWrVtQKpW4efMm6tatK7v67FG9epjCPrBH9anPHtWrhynsgyn04GkjIiIikhWGFyIiIpIVhhciIiKSFYYXAAqFApGRkVAoFLKszx7Vq4cp7AN7VJ/67FG9epjCPphCDy7YJSIiIlnhzAsRERHJCsMLERERyQrDCxEREckKwwsRERHJSo0PLytWrECTJk1gZWWFjh074tChQ5LW//XXX9G3b184OztDEARs375d0vpRUVHo1KkTbG1t4eDggH79+iEjI0PSHitXrkT79u1Rt25d1K1bF76+vtizZ4+kPR4XFRUFQRAQHh4uWc05c+ZAEASdl5OTk2T1H7l06RLefPNNNGjQALVr18YzzzyD5ORkyeo3bty4xH4IgoDQ0FDJejx48ACzZ89GkyZNYG1tjaZNm+KDDz6ARqORrMft27cRHh4Od3d3WFtbo0uXLjh27FiF6z3pOBNFEXPmzIGzszOsra3Ro0cPnDx5UtIeP/zwA3r37g17e3sIgoDU1FRJ96OwsBARERFo164dbGxs4OzsjOHDh+Off/6RdD/mzJkDDw8P2NjYoH79+vD398eRI0ck7fG4d955B4IgYOnSpZLVHzFiRIljpHPnzpLvQ3p6Ol599VUolUrY2tqic+fOuHDhgmQ9SjvWBUHAxx9/LFmPO3fuYPz48XBxcYG1tTVat26NlStX6l1fnx7//vsvRowYAWdnZ9SuXRsBAQHIzMw0qEdxNTq8fPvttwgPD8esWbNw/PhxPP/88+jTp49B//M9yd27d+Hl5YXly5dLVvNx8fHxCA0NRWJiImJjY/HgwQP06tULd+/elayHi4sLoqOjkZSUhKSkJPTs2ROBgYEG/+Wvj2PHjmHVqlVo37695LXbtGmDnJwc7SstLU3S+tevX0fXrl1haWmJPXv24NSpU1i0aBHq1asnWY9jx47p7ENsbCwA4PXXX5esx8KFC/H5559j+fLlSE9Px0cffYSPP/4Yn332mWQ9Ro0ahdjYWGzcuBFpaWno1asX/P39cenSpQrVe9Jx9tFHH2Hx4sVYvnw5jh07BicnJ7z00ku4ffu2ZD3u3r2Lrl27Ijo6ukL78KQe9+7dQ0pKCt577z2kpKTghx9+wOnTp/Hqq69K1gMAWrZsieXLlyMtLQ0JCQlo3LgxevXqhcuXL0vW45Ht27fjyJEjcHZ2lnQfACAgIEDnWNm9e7ekPf7++29069YNHh4eOHjwIE6cOIH33nsPVlZWkvV4fPw5OTmIiYmBIAgYOHCgZD0mTZqEvXv3YtOmTUhPT8ekSZMwYcIE/Pjjj5L0EEUR/fr1w9mzZ/Hjjz/i+PHjcHd3h7+/f+X+nZL8aUky8uyzz4pjxozR2ebh4SHOmDHDKP0AiNu2bTNK7Ufy8vJEAGJ8fLxR+9SvX19cs2aNpDVv374ttmjRQoyNjRX9/PzEsLAwyWpHRkaKXl5ektUrTUREhNitWzej9iguLCxMbNasmajRaCSr+corr4ghISE62wYMGCC++eabktS/d++eaG5uLu7cuVNnu5eXlzhr1qxK1y9+nGk0GtHJyUmMjo7WbisoKBCVSqX4+eefS9LjcVlZWSIA8fjx4xWqrU+PR44ePSoCEM+fP2+0Hjdv3hQBiAcOHJC0x8WLF8VGjRqJf/75p+ju7i4uWbJEsvrBwcFiYGBgherp22PQoEGSHRNl9SguMDBQ7Nmzp6Q92rRpI37wwQc627y9vcXZs2dL0iMjI0MEIP7555/abQ8ePBDt7OzE1atXV6iHKIpijZ15uX//PpKTk9GrVy+d7b169cLhw4eraFSVd/PmTQCAnZ2dUeoXFRVh8+bNuHv3Lnx9fSWtHRoaildeeQX+/v6S1n0kMzMTzs7OaNKkCd544w2cPXtW0vo7duyAj48PXn/9dTg4OKBDhw5YvXq1pD0ed//+fWzatAkhISEQBEGyut26dcPPP/+M06dPAwBOnDiBhIQEvPzyy5LUf/DgAYqKikr8hGptbY2EhARJejwuKysLubm5Ose6QqGAn5+frI914OHxLgiCpLN7j7t//z5WrVoFpVIJLy8vyepqNBoMGzYM06ZNQ5s2bSSr+7iDBw/CwcEBLVu2xNtvv428vDzJams0GuzatQstW7ZE79694eDggOeee07yZQGP+/fff7Fr1y6MHDlS0rrdunXDjh07cOnSJYiiiLi4OJw+fRq9e/eWpL5arQYAnePd3NwctWrVqtTxXmPDy5UrV1BUVARHR0ed7Y6OjsjNza2iUVWOKIqYPHkyunXrhrZt20paOy0tDXXq1IFCocCYMWOwbds2eHp6SlZ/8+bNSElJQVRUlGQ1H/fcc8/hyy+/xL59+7B69Wrk5uaiS5cuuHr1qmQ9zp49i5UrV6JFixbYt28fxowZg4kTJ+LLL7+UrMfjtm/fjhs3bmDEiBGS1o2IiMDgwYPh4eEBS0tLdOjQAeHh4Rg8eLAk9W1tbeHr64t58+bhn3/+QVFRETZt2oQjR44gJydHkh6Pe3Q8m9KxDgAFBQWYMWMGhgwZIvkTe3fu3Ik6derAysoKS5YsQWxsLOzt7SWrv3DhQlhYWGDixImS1Xxcnz598NVXX+GXX37BokWLcOzYMfTs2VP7D2ll5eXl4c6dO4iOjkZAQAD279+P/v37Y8CAAYiPj5ekR3EbNmyAra0tBgwYIGndTz/9FJ6ennBxcUGtWrUQEBCAFStWoFu3bpLU9/DwgLu7O959911cv34d9+/fR3R0NHJzcyt1vFtIMjoZK/4TqyiKkv4U+zSNHz8ef/zxh1F+em3VqhVSU1Nx48YNbN26FcHBwYiPj5ckwGRnZyMsLAz79+836HyxIfr06aP9dbt27eDr64tmzZphw4YNmDx5siQ9NBoNfHx8sGDBAgBAhw4dcPLkSaxcuRLDhw+XpMfj1q5diz59+hi8XuBJvv32W2zatAlff/012rRpg9TUVISHh8PZ2RnBwcGS9Ni4cSNCQkLQqFEjmJubw9vbG0OGDEFKSook9UtjSsd6YWEh3njjDWg0GqxYsULy+i+88AJSU1Nx5coVrF69GkFBQThy5AgcHBwqXTs5ORnLli1DSkqK0f78Bw0apP1127Zt4ePjA3d3d+zatUuSf/wfLV4PDAzEpEmTAADPPPMMDh8+jM8//xx+fn6V7lFcTEwMhg4dKvnfkZ9++ikSExOxY8cOuLu749dff8W4ceOgUqkkmQW3tLTE1q1bMXLkSNjZ2cHc3Bz+/v46fydXRI2debG3t4e5uXmJn7zy8vJK/IQmBxMmTMCOHTsQFxcHFxcXyevXqlULzZs3h4+PD6KiouDl5YVly5ZJUjs5ORl5eXno2LEjLCwsYGFhgfj4eHz66aewsLBAUVGRJH0eZ2Njg3bt2lV6xfvjVCpViTDXunVrSReAP3L+/HkcOHAAo0aNkrz2tGnTMGPGDLzxxhto164dhg0bhkmTJkk6K9asWTPEx8fjzp07yM7OxtGjR1FYWIgmTZpI1uORR1eVmcqxXlhYiKCgIGRlZSE2NlbyWRfg4fHRvHlzdO7cGWvXroWFhQXWrl0rSe1Dhw4hLy8Pbm5u2uP9/PnzmDJlCho3bixJj+JUKhXc3d0lO97t7e1hYWHx1I73Q4cOISMjQ/LjPT8/HzNnzsTixYvRt29ftG/fHuPHj8egQYPwySefSNanY8eO2h9+c3JysHfvXly9erVSx3uNDS+1atVCx44dtVdrPBIbG4suXbpU0agMJ4oixo8fjx9++AG//PKLUf7yL6uvVFOwL774ItLS0pCamqp9+fj4YOjQoUhNTYW5ubkkfR6nVquRnp4OlUolWc2uXbuWuEz99OnTcHd3l6zHI+vWrYODgwNeeeUVyWvfu3cPZma6fzWYm5tLeqn0IzY2NlCpVLh+/Tr27duHwMBAyXs0adIETk5OOsf6/fv3ER8fL6tjHfhfcMnMzMSBAwfQoEGDp9JXyuN92LBh+OOPP3SOd2dnZ0ybNg379u2TpEdxV69eRXZ2tmTHe61atdCpU6endryvXbsWHTt2lHTdEfDw/6fCwsKndrwrlUo0bNgQmZmZSEpKqtTxXqNPG02ePBnDhg2Dj48PfH19sWrVKly4cAFjxoyRrMedO3dw5swZ7e+zsrKQmpoKOzs7uLm5Vbp+aGgovv76a/z444+wtbXV/nSpVCphbW1d6foAMHPmTPTp0weurq64ffs2Nm/ejIMHD2Lv3r2S1Le1tS2xRsfGxgYNGjSQbO3O1KlT0bdvX7i5uSEvLw8ffvghbt26JdlpEODhJYddunTBggULEBQUhKNHj2LVqlVYtWqVZD2Ah1PW69atQ3BwMCwspD+E+/bti/nz58PNzQ1t2rTB8ePHsXjxYoSEhEjWY9++fRBFEa1atcKZM2cwbdo0tGrVCm+99VaF6j3pOAsPD8eCBQvQokULtGjRAgsWLEDt2rUxZMgQyXpcu3YNFy5c0N535dE/bE5OTnrfU6i8Hs7OznjttdeQkpKCnTt3oqioSHu829nZoVatWpXu0aBBA8yfPx+vvvoqVCoVrl69ihUrVuDixYsGXY7/pD+r4qHL0tISTk5OaNWqVaXr29nZYc6cORg4cCBUKhXOnTuHmTNnwt7eHv3795dsH6ZNm4ZBgwahe/fueOGFF7B371789NNPOHjwoGQ9AODWrVv47rvvsGjRIr3rGtLDz88P06ZNg7W1Ndzd3REfH48vv/wSixcvlqzHd999h4YNG8LNzQ1paWkICwtDv379SlwwY5AKX6dkIv7zn/+I7u7uYq1atURvb2/JLzGOi4sTAZR4BQcHS1K/tNoAxHXr1klSXxRFMSQkRPtn1LBhQ/HFF18U9+/fL1n90kh9qfSgQYNElUolWlpais7OzuKAAQPEkydPSlb/kZ9++kls27atqFAoRA8PD3HVqlWS99i3b58IQMzIyJC8tiiK4q1bt8SwsDDRzc1NtLKyEps2bSrOmjVLVKvVkvX49ttvxaZNm4q1atUSnZycxNDQUPHGjRsVrvek40yj0YiRkZGik5OTqFAoxO7du4tpaWmS9li3bl2p70dGRkrS49El2KW94uLiJOmRn58v9u/fX3R2dhZr1aolqlQq8dVXXxWPHj0q6Z9VcYZeKl1e/Xv37om9evUSGzZsKFpaWopubm5icHCweOHCBcn3Ye3atWLz5s1FKysr0cvLS9y+fbvkPb744gvR2tq6wsfHk3rk5OSII0aMEJ2dnUUrKyuxVatW4qJFiwy6/cKTeixbtkx0cXHR/veYPXt2pf8+EURRFCucfIiIiIieshq75oWIiIjkieGFiIiIZIXhhYiIiGSF4YWIiIhkheGFiIiIZIXhhYiIiGSF4YWIiIhkheGFiCQ3Z84cPPPMM1U9DCIyUQwvRCZgxIgREAQBgiDA0tISTZs2xdSpU3H37t1K1a1oCJk6dSp+/vnnSvUmIipLjX62EZEpCQgIwLp161BYWIhDhw5h1KhRuHv3LlauXFnis4WFhbC0tDTaWOrUqYM6deoYrT4R1WyceSEyEQqFAk5OTnB1dcWQIUMwdOhQbN++HcD/ZlBiYmLQtGlTKBQKiKKICxcuIDAwEHXq1EHdunURFBSEf//9FwCwfv16zJ07FydOnNDO6qxfvx4AcPPmTYwePRoODg6oW7cuevbsiRMnTmjHUnzGZsSIEejXrx8++eQTqFQqNGjQAKGhoSgsLCxzf06cOIEXXngBtra2qFu3Ljp27IikpCTt+4cPH0b37t1hbW0NV1dXTJw4UWemKS8vD3379oW1tTWaNGmCr776Co0bN8bSpUsBAOfOnYMgCEhNTdV+58aNGxAEQefheqdOncLLL7+MOnXqwNHREcOGDcOVK1e07/fo0QMTJ07E9OnTYWdnBycnJ8yZM0dnX27cuIHRo0fD0dERVlZWaNu2LXbu3Kn3vhCRLoYXIhNlbW2tEw7OnDmDLVu2YOvWrdp/sPv164dr164hPj4esbGx+PvvvzFo0CAAwKBBgzBlyhS0adMGOTk5yMnJwaBBgyCKIl555RXk5uZi9+7dSE5Ohre3N1588UVcu3atzPHExcXh77//RlxcHDZs2ID169drw1Bphg4dChcXFxw7dgzJycmYMWOGdrYoLS0NvXv3xoABA/DHH3/g22+/RUJCAsaPH6/9/ogRI3Du3Dn88ssv+P7777FixQrk5eUZ9GeYk5MDPz8/PPPMM0hKSsLevXvx77//IigoSOdzGzZsgI2NDY4cOYKPPvoIH3zwAWJjYwE8fAp4nz59cPjwYWzatAmnTp1CdHQ0zM3N9d4XIiqmUo91JKJqITg4WAwMDNT+/siRI2KDBg3EoKAgURRFMTIyUrS0tBTz8vK0n9m/f79obm6u87TdkydPigC0TxGOjIwUvby8dHr9/PPPYt26dcWCggKd7c2aNRO/+OKLUr8XHBwsuru7iw8ePNBue/3118VBgwaVuU+2trbi+vXrS31v2LBh4ujRo3W2HTp0SDQzMxPz8/PFjIwMEYCYmJiofT89PV0EoH168aOnNB8/flz7mevXr+s8pfm9994Te/XqpdMnOztb56nefn5+Yrdu3XQ+06lTJzEiIkIUxYdPATczMyvzKeBP2hciKolrXohMxM6dO1GnTh08ePAAhYWFCAwMxGeffaZ9393dHQ0bNtT+Pj09Ha6urnB1ddVu8/T0RL169ZCeno5OnTqV2ic5ORl37txBgwYNdLbn5+fj77//LnN8bdq00c42AIBKpUJaWlqZn588eTJGjRqFjRs3wt/fH6+//jqaNWumHcOZM2fw1VdfaT8viiI0Gg2ysrJw+vRpWFhYwMfHR/u+h4cH6tWrV2a/svY1Li6u1PU7f//9N1q2bAkAaN++vc57KpVKO8uTmpoKFxcX7WdL61HevrRu3dqgMRPVBAwvRCbihRdewMqVK2FpaQlnZ+cSC3JtbGx0fi+KIgRBKFGnrO2PaDQaqFQqnXUhj5QXDoqPRxAEaDSaMj8/Z84cDBkyBLt27cKePXsQGRmJzZs3o3///tBoNHjnnXcwceLEEt9zc3NDRkaGtkdZzMwenjUXRVG7rfgaHI1Gg759+2LhwoUlvq9SqfTaN2tr6zLH8KhHeftCRCUxvBCZCBsbGzRv3lzvz3t6euLChQvIzs7Wzr6cOnUKN2/e1P60X6tWLRQVFel8z9vbG7m5ubCwsEDjxo0lG39pWrZsiZYtW2LSpEkYPHgw1q1bh/79+8Pb2xsnT54sc39bt26NBw8eICkpCc8++ywAICMjAzdu3NB+5tEsVE5ODjp06AAAOot3gYf7unXrVjRu3BgWFhX767J9+/a4ePEiTp8+Xersy5P2hYhK4oJdohrK398f7du3x9ChQ5GSkoKjR49i+PDh8PPz055uady4MbKyspCamoorV65ArVbD398fvr6+6NevH/bt24dz587h8OHDmD17ts7VQJWRn5+P8ePH4+DBgzh//jx+++03HDt2TBuqIiIi8PvvvyM0NBSpqanIzMzEjh07MGHCBABAq1atEBAQgLfffhtHjhxBcnIyRo0apTMLYm1tjc6dOyM6OhqnTp3Cr7/+itmzZ+uMIzQ0FNeuXcPgwYNx9OhRnD17Fvv370dISEiJUFcWPz8/dO/eHQMHDkRsbCyysrKwZ88e7N27V699IaKSGF6IaihBELB9+3bUr18f3bt3h7+/P5o2bYpvv/1W+5mBAwciICAAL7zwAho2bIhvvvkGgiBg9+7d6N69O0JCQtCyZUu88cYbOHfuHBwdHSUZm7m5Oa5evYrhw4ejZcuWCAoKQp8+fTB37lwAD2cz4uPjkZmZieeffx4dOnTAe++9p3MqZ926dXB1dYWfnx8GDBigvbT7cTExMSgsLISPjw/CwsLw4Ycf6rzv7OyM3377DUVFRejduzfatm2LsLAwKJVK7WknfWzduhWdOnXC4MGD4enpienTp2vDjz77QkS6BPHxE75ERCascePGCA8PR3h4eFUPhYgqgTMvREREJCsML0RERCQrPG1EREREssKZFyIiIpIVhhciIiKSFYYXIiIikhWGFyIiIpIVhhciIiKSFYYXIiIikhWGFyIiIpIVhhciIiKSFYYXIiIikpX/B+lcJaVQkDgbAAAAAElFTkSuQmCC",
      "text/plain": [
       "<Figure size 640x480 with 1 Axes>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "import seaborn as sns\n",
    "\n",
    "fig = sns.heatmap(\n",
    "    one_hot_encoded_sequence,\n",
    "    square=True,\n",
    "    cbar=False,\n",
    "    cmap=\"inferno\")\n",
    "\n",
    "fig.set_ylabel(\"Amino Acid Index\")\n",
    "fig.set_xlabel(\"Protein sequence\")"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "default",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.12.11"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
